---
title: Nitric-oxide-feedback to ciliary photoreceptor cells orchestrates UV avoidance
  in zooplankton
author: 
  - name: "Kei Jokura<sup>1,`*`</sup>, Nobuo Ueda<sup>2</sup>, Martin Gühmann<sup>3</sup>, Luis Alfonso Yañez-Guerra<sup>1</sup>, Piotr Słowiński<sup>1</sup>, Kyle C. A. Wedgewood<sup>1</sup> and Gáspár Jékely<sup>1,4,`*`</sup>"
    affiliation: "<sup>1</sup>Living Systems Institute, University of Exeter, Stocker road, Exeter, EX4 4QD, United Kingdom <sup>2</sup>Okinawa Institute of Science and Technology, Okinawa, Japan, <sup>3</sup>School of Biological Sciences, University of Bristol, Bristol, United Kingdom, <sup>4</sup>Centre for Organismal Studies (COS), University of Heidelberg, 69120 Heidelberg, Germany  <br> <sup>*</sup>Correspondence`:` kei.jokura @ exeter.ac.uk, gaspar.jekely @ cos.uni-heidelberg.de"
output:
  html_document:
    theme: lumen
  pdf_document:
    latex_engine: lualatex
    template: default
  word_document: default
geometry: left=0.8in,right=0.8in,top=0.8in,bottom=0.5in
mainfont: Arial
monofont: Times New Roman
documentclass: article
classoption: onecolumn
linestretch: 1.2
fontsize: 10pt
bibliography: references.bib
csl: elife.csl
---


```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)

```


## Abstract

Nitric oxide (NO) generated by nitric-oxide synthase (NOS) is a key regulator of animal physiology. Here we uncover a role for NO in shaping circuit dynamics to orchestrate light-avoidance behaviour. We studied UV-avoidance mediated by brain ciliary photoreceptors (cPRCs) in larval *Platynereis dumerilii*, a marine annelid. We found NOS specifically expressed in interneurons (INNOS) postsynaptic to cPRCs. Stimulation of cPRCs by violet light leads to cPRC inhibition but concomitant INNOS activation and NO production. NO feeds back to cPRCs and triggers their delayed activation through an unconventional guanylyl cyclase. This results in the activation of projection interneurons and the inhibition of serotonergic ciliomotor neurons. In *NOS* mutants, NO feedback and projection-neuron activation do not occur and avoidance behaviour is defective. By mathematical modelling, we could capture the circuit dynamics in both wild-type and mutant larvae. Our results reveal how NO-mediated neuroendocrine signalling gates a synaptic circuit to initiate light-avoidance behaviour.

## Introduction

Neuronally produced nitric oxide (NO) can act as a diffusible neuromodulator. This small free-radical molecule is synthesized by nitric oxide synthase (NOS)...

In the mammalian retina, NO is produced by amacrine cells and activates a *soluble guanylate cyclase (sGC)* in cone bipolar cells. The NO-dependent increased in cGMP tunes bipolar cell activity .... [refs]
[other examples]

To investigate NO signalling in the modulation of neural circuit dynamics, we studied the larval stages of the marine annelid model *Platynereis dumerilii* [@ozpolat2021]. We uncovered an essential function for NO in these larvae in UV/violet-light avoidance.
In *Platynereis* larvae, UV/violet avoidance is characterised by downward swimming and is mediated by the UV/violet-responding brain ciliary photoreceptor cells (cPRCs).

The cPRCs express a vertebrate-type ciliary opsin, c-opsin1 [@Arendt2004]. This opsin forms a UV-absorbing bistable photopigment with an absorption maximum around 384 nm [@tsukamoto2017, @Veraszto2018]. *c-opsin1* knockout larvae show defective UV/violet avoidance [@Veraszto2018]. 
Here we show that *Platynereis* *NOS* is expressed in interneurons of the cPRC circuitry and is required for UV/violet-avoidance behaviour. By combining calcium imaging across the fully-mapped cPRC circuit, genetic perturbations and mathematical modelling, we describe how NO tunes circuit dynamics through non-synaptic reverse signalling to cPRCs. This delayed neuroendocrine feedback is essential for circuit output, and integrates UV/violet input and also induces a short-term memory manifest in altered circuit activity.

#In adult worms, UV light and c-opsin1 modulate locomotion and maturation [@Veedin2021]. 



## Results

**Nitric oxide synthase is expressed in interneurons of the UV-avoidance circuit**

We identified a single *nitric oxide synthase* (*NOS*) gene in the *Platynereis dumerilii* genome and transcriptome data. Phylogenetic analysis of NOS proteins indicate that *Platynereis* NOS belongs to an orthology group of metazoan NOS sequences **(Figure 1---figure supplement 1)**. To characterise the expression pattern of *NOS* we used in situ hybridization chain reaction (HCR) and transient transgenesis. In two- and three-day-old larvae, we detected *NOS* expression in four cells (two of them weakly expressing) in the apical organ region **(Figure 1D and Figure 1---figure supplement 2)**. *NOS* was also expressed in the region of the visual eyes (adult eyes) and the pigmented eyespots **(Figure 1---figure supplement 1)**. The four apical organ cells, but not the eyes, were also labelled with a *NOS* reporter construct driving palmitoylated-tdTomato **(Figure 1E)**. This reporter also revealed the axonal projections of these central NOS-expressing neurons. The position and morphology of the four *NOS+* cells allowed us to identify the same four cells as four interneurons (INNOS) in our three-day-old whole-body *Platynereis* volume EM data [@williams2017; @veraszto2020] **(Figure 1B,C)**. In the synaptic connectome, the INNOS cells are postsynaptic to the UV-sensory cPRCs and presynaptic to the INRGW interneurons, which are also cPRC targets **(Figure 1C,F)**. INRGW neurons synapse on the head serotonergic ciliomotor neurons (Ser-h1), which synapse on the prototroch ciliary band and the cholinergic MC ciliomotor neurons **(Figure 1C,F)**.

```{r echo=FALSE,fig.cap="**Figure 1. Identification of NOS-expressing interneurons (INNOS) within the cPRC circuit.** **(A)** SEM image of a three-day-old Platynereis larva. **(B, C)** Five types of neurons (cPRC, INNOS, INRGW, Ser-h1 and MC) in the cPRC circuit reconstructed in the whole-body transmission electron microscopy (ssTEM) dataset of a three-day-old larva. Ciliated cells labelled with (B) are shown in grey. The INNOS on the ventral side labelled with (C) are shown in purple. Axons and dendrites appear as lines and cell body positions are represented by spheres. **(D)** Expression analysis of the NOS gene (magenta) using in situ HCR. Anterior view of the larva at two-day-old. Co-stained image using acetylated α-tubulin antibody (acTub: green). **(E)** Immunostaining with anti-HA antibody against HA tagged palmitoylated-tdTomato expressed under the upstream of the start site of NOS (NOSp::palmi-3xHA: magenta). Co-staining with acetylated α-tubulin antibody as marker for cilia of cPRC and axonal scaffold (acTub: green). Views of larvae at three-day-old from the anterior side. Insert: One of the reconstructed INNOS is shown from the anterior side. **(F)** Wiring diagram of the cPRC circuit. Hexagons and arrows represent cell groups and their synaptic connections, respectively. Numbers inside hexagons indicate the number of cells grouped in each node. Arrow line thickness (synapse weight) is equal to the number of synapses."}

knitr::include_graphics(path = "figures/Fig1.png")

```



**Nitric oxide produced during UV/violet stimulation gates the output of the cPRC circuit**

The expression of *NOS* in the INNOS interneurons in the cPRC circuit suggests that NO signalling may be involved in UV/violet-avoidance. To test this, first we asked whether NO is produced during UV/violet stimulation of the larvae. We injected the fluorescent NO-reporter DAF-FM into zygotes and imaged two-day-old larvae while exposing the ramified sensory cilia of the cPRCs to 405 nm violet light. Following light stimulation of the cPRCs but not a control area, we detected an increase in DAF-FM fluorescence in the anterior neurosecretory neuropil, the region of INNOS projections **(Figure 2)**.

```{r echo=FALSE,fig.cap="**Figure 2. NO produced by UV/violet stimulation to cPRCs.** **(A)** A NO-positive cell visualised by DAF-FM (green) is shown (dashline). The white line indicates the outer frame of the larva. All subsequent images are of two-day-old larvae. **(B)** Intensity changes before (upper) and during 405 nm light stimulation (lower) in a cell visualised by DAF-FM. **(C)** The intensity of DAF-FM changes over time when cPRC (green) and other cells (ctr: gray) are stimulated with UV/violet (405 nm) light. Purple squares indicate the timing of 405 nm light stimulation, the transparent lines show the optical responses to individual intensities, each intensities are normalized (ΔF/F0) and the dark trace lines and the shaded bands show the mean value and the standard error range of the mean."}

knitr::include_graphics(path = "figures/Fig2.png")

```


**Nitric oxide signalling mediates UV-avoidance behaviour**

Next, we wanted to directly test whether NO signalling is involved in UV/violet avoidance. To achieve this, we generated two *Platynereis* *NOS* knockout lines with the CRISPR/Cas9 method. We recovered two deletions (NOSΔ11/Δ11 and Δ23/Δ23), both leading to frame-shift and an early stop codon and thus likely representing null alleles **(Figure 3A)**. We could establish a homozygous line for both mutations indicating that NOS is not an essential gene. To quantify UV avoidance, we recorded the trajectories of freely swimming wild type and mutant larvae in vertical columns, illuminated laterally from two opposite sides with 395 nm UV light **(Figure 3B, C and Figure 3---figure supplement 1A-D)**. As previously shown, wild-type larvae swim downward following non-directional UV/violet light stimulation [@Veraszto2018]. In contrast, both two- and three-day-old homozygous *NOS*-mutant larvae showed a strongly diminished UV-avoidance response **(Figure 3B-D and Figure 3---figure supplement 1)**. This phenotype is similar to the defective UV-avoidance of *c-opsin1* mutant larvae and reveals a requirement for NOS in UV-avoidance behaviour [@Veraszto2018]. Wild type but not mutant larvae also showed an increase in swimming speed under UV light that may be due to downward swimming trajectories (swimming with gravity) **(Figure 3---figure supplement 1C,D)**. 
We also tested directional phototaxis, by exposing larvae to 480 nm directional and collimated light from the top of the column. Three-day-old but not two-day-old *NOS*-mutant larvae also showed reduced phototactic behaviour, suggesting a function for *NOS* in the visual eyes that mediate three-day-old phototaxis [@Randel2014a] **(Figure 1---figure supplement 1A,B)**. 
To distinguish between an acute or developmental function of NOS in light responses, we next tested larvae exposed to the NOS inhibitor L-NAME. Larvae incubated for 5 min in 0.1 mM or 1 mM L-NAME showed a dose-dependent inhibition of UV avoidance. In contrast, phototaxis was not effected **(Figure 3E)**. Overall, our results indicate an acute requirement for NOS signalling in UV-avoidance and a possible indirect, developmental role in visual phototaxis. 

```{r echo=FALSE,fig.cap="**Figure 3. Larvae with inhibited NOS have a delayed response to UV-avoidance.** **(A)** Top: The domain and Exon/Intron structure of NOS. Bottom: Close-up region showing the genomic locus of NOS gene and the wild-type sequence (WT) targeted by CRISPR/Cas9. The generated mutants (NOSΔ11, NOSΔ23) are also shown. Pink indicates target sites. Gray shows PAM sequences, red shows stop codons. **(B)** Overlaid trajectories for WT (n=32) and NOS mutant (NOSΔ11/Δ11, n=26 and NOSΔ23/Δ23, n=47) at three-day-old larvae. 0 sec as the starting point. After 10 sec, UV (395 nm) stimulation from the side. **(C)** The temporal changes in the vertical position of the WT and mutant larvae before (dark) and after UV stimulation (side 395 nm) are shown. The starting points of each larval trajectory are set to 0. **(D)** Vertical swimming in wild-type (WT) and mutant (NOSΔ11 and NOSΔ23) larvae at three-day-old stimulated with UV (395 nm) light from side, blue (488 nm) light from top and UV (395 nm) light from top. The data are shown in 30 s bins. **(E)** Vertical swimming in larvae treated with NOS inhibitors, L-NAME at three-day-old stimulated with UV (395 nm) light from side, blue (488 nm) light from top and UV (395 nm) light from top. The data are shown in 30 s bins."}

knitr::include_graphics(path = "figures/Fig3.png")

```

**NO feedback tunes cPRC responses to UV/violet stimulation**

To investigate how NO signalling alters the dynamics of the cPRC circuit, we carried out calcium imaging experiments. We ubiquitously expressed the calcium sensor GCaMP6s in larvae and imaged calcium signals during 405 nm stimulation of the cPRCs. As we have shown previously, a 20-sec local stimulation of cPRC cilia lead to a transient increase in cPRC calcium levels, followed by a transient decrease [@Veraszto2018]. After ~20-sec, calcium levels in cPRCs were raising again, reaching higher levels than at the start of the stimulus -- a response that may involve depolarisation. This activation phase occurs after the 20 sec stimulation period and is likely due to a delayed neuroendocrine feedback [@Veraszto2018]. To determine whether NO mediates such a feedback, we repeated the experiment in *NOS*-mutant larvae. While we detected the initial activation phase followed by inhibition, in *NOS*-mutants this was not followed by delayed activation and calcium levels dropped to a low steady-state level **(Figure 4E,F)**. We thus identified a requirement for NO signalling in the late-phase activation of cPRCs.

**Two unconventional guanylyl cyclases are expressed in the cPRCs**

Next, we aimed to identify the NO-receptor in the cPRCs. NO generally acts via soluble guanylate cyclases (sGC), belonging to the guanylate cyclase family with a CYC domain (PFAM domain: PF00211). NO binding to the heme group of sGC leads to increased cyclic guanosine monophosphate (cGMP) production. Analysis of sGCs in *Platynereis* indicated that these genes are not expressed in any of the cells of the cPRC circuit [@Veraszto2017]. Recently, Moroz and coworkers reported an atypical but widely conserved family of guanylyl cyclases with a NIT (nitrite/nitrate sensing) domain (PF08376) (NIT-GC) as potential mediators of NO signalling [@Moroz2020]. To search for NIT-GCs in *Platynereis*, we searched transcriptome resources and retrieved 15 potential NIT-GC homologs **(Figure 4---figure supplement 1)**. To analyse the relationship of these sequences to metazoan NIT-GCs, we retrieved protein sequences with a CYC domain from the transcriptome and genome databases of 45 metazoan species and 2 choanoflagellate species. We carried out cluster analysis and did phylogenetic reconstruction on a group of membrane-bound guanylyl cyclases with sGCs as an outgroup. In agreement with Moroz et al. [@Moroz2020], we found a group of GCs with NIT domains with representatives in placozoans, cnidarians, some ecdysozoans, echinoderms, and lophotrochozoans. The 15 *Platynereis* sequences were parts of several major and ancient clades of the tree **(Figure 4---figure supplement 1)**. To characterise the expression of NIT-GCs, we used previously published spatially mapped single-cell transcriptome data [@Achim2015; @williams2017]. Among the 15 NIT-GCs, two showed high and specific expression in the cPRCs and one was expressed in the INNOS cells **(Figure 4---figure supplement 1)**. In the single-cell data, we could identify the cPRCs by the specific expression of c-opsin1 and a pedal-peptide2 neuropeptide precursor (MLD proneuropeptide) [@Arendt2004; @williams2017], which have previously been described as cPRC markers **(Figure 4---figure supplement 2A)**. The INNOS cells were identified by NOS expression and spatial mapping in the brain [@Achim2015]. We decided to focus on two NIT-GCs expressed in the cPRCs and with a full-length sequence, NIT-GC1 and NIT-GC2. To confirm the single-cell data, we first carried out in situ HCR with probes for *NIT-GC1* and *NIT-GC2* mRNA. Both genes were specifically expression in the four cPRCs, as confirmed by co-labeling with an acetylated α-tubulin antibody and with an HCR probe against *pedal peptide 2/MLD proneuropeptide* **(Figure 4A,B and Figure 4---figure supplement 2A-C)**. 
To analyse the subcellular localisation of NIT-GC1 and NIT-GC2 at the protein level, we raised and affinity-purified polyclonal antibodies against a specific peptide sequence from both proteins. In immunostainings, we found that NIT-GC1 was localise to the region corresponding to the axonal projections of the cPRCs in the anterior nerve plexus **(Figure 4C)**. Co-immunostaining with the rabbit NIT-GC1 antibody and a custom rat antibody raised against a fragment of *Platynereis* NOS, revealed that both proteins were localised in close proximity in the neurosecretory plexus **(Figure 4---figure supplement 2D)**. In contrast, NIT-GC2 specifically labelled the large sensory ciliary region of the cPRCs **(Figure 4D)**. These different subcellular localisations suggest that the two NIT-GCs are involved in different intracellular signalling processes in the ciliary and axonal regions of the cPRCs.


**NIT-GC1 produces cGMP in an NO-dependent manner**

To further characterise these atypical guanylyl cyclases, we focused on NIT-GC1 and carried out in vitro experiments. In bacteria, NIT domains are thought to regulate cellular functions in response to intra- or extracellular nitrate and nitrite. NIT-GC1 has a NIT domain and a highly conserved cyclase domain that is expected to catalyse cGMP synthesis **(Figure 4I)**. The NIT domain may render NIT-GC1 dependent on NO signals. To test this, we co-expressed the synthetic cGMP indicator Green cGull [@Matsuda2017] and NIT-GC1 in cultured cells COS-7 (monkey kidney). For balanced expression, we used a single plasmid with the two open-reading frames separated by the 2A self-cleaving peptide **(Figure 4J)**. Application of the NO donor SNAP lead to increased cGMP levels, an effect we did not observe when cells were exposed to DMSO or when Green cGull was expressed alone **(Figure 4K-M)**. To test whether cGMP production is dependent on the NIT domain, we also tested a deletion construct of NIT-GC1 lacking the NIT domain **(Figure 4I)**. Cells expressing this construct and Green cGull did not show increased cGMP levels when exposed to SNAP **(Figure 4N)**. These results indicate that NIT-GC1 is able to catalyse cGMP production in an NO-dependent manner and this function requires the NIT domain. These results establish NIT-GC1 as a biochemical NO sensor. 

**NIT-GC1 is required for NO-mediated feedback to cPRCs during the UV response**

To test the in vivo function of NIT-GC1 and NIT-GC2 in cPRC responses, we combined calcium imaging with morpholino-mediated knockdowns. We used two translation-blocking morpholinos for each *NIT-GC* gene and tested knockdown efficiency by immunostaining injected animals with the NIT-GC1 and NIT-GC2 antibodies **(Figure 4---figure supplement 2E,F)**. For both genes, the morpholinos led to a strong reduction in the respective antibody signal, confirming efficient knockdown and antibody specificity.

In NIT-GC1 morphant larvae, the delayed activation of cPRCs following 405 nm stimulation did not occur **(Figure 4G)**. This phenotype is similar to the phenotype of *NOS* mutants suggesting that NIT-GC1 acts as the NO sensor in cPRCs to drive their delayed activation. This could occur via increased cGMP production and the opening of a cyclic-nucleotide-gated (CNG) channel specific to cPRCs. 
NIT-GC2 morphant larvae, in contrast, showed a step-up increase in calcium following light stimulation **(Figure 4H)**. The calcium signal decayed during stimulation and was off after light off. These data support an essential role for ciliary-localised NIT-GC2 in suppressing cPRC calcium following its transient rise at stimulus onset. 
Overall, these knockdown experiments revealed different signalling mechanisms for the two NIT-GCs that may be due to their different subcellular localisations.


```{r echo=FALSE,fig.cap="**Figure 4. cPRC-specific expression of NIT-GC1 localises in close proximity to NOS in the ANS region.** **(A, B)** Expression analysis of the NIT-GC1 (C) and NIT-GC2 (D) gene (magenta) using in situ HCR. Co-staining image using acetylated α-tubulin antibody (ac α-tub: green). **(C, D)** Localisation analysis using NIT-GC1 and NIT-GC2 antibodies. Green shows co-staining with acetylated α-tubulin antibody (ac α-tub). **(E, F)** Changes over time in GCaMP6s intensities in cPRC when stimulated with UV in WT (dark yellow) and NOS mutant (E: NOSΔ11/Δ11, F: NOSΔ23/Δ23) (light pink) larvae. **(G, H)** Changes over time in GCaMP6s intensities in cPRC when stimulated with UV in NIT-GC1 (G: NIT1) and NIT-GC2 (H: NIT2) knockdown by morpholino (MO1 and 2) larvae. **(I)** The domain structure of NIT-GC1. Below shows the mutant NIT-GC1 (ΔNIT-GC1) with deletion of the NIT domain. Grey squares indicate the transmembrane region (TM). **(J)** A schematic of the NIT-GC1 assay; COS7 cells were used to express NIT-GC1 and the cGMP indicator Green cGull, and the NO donor SNAP and DMSO as control. Green cGull shows cGMP-dependent green fluorescence. **(K-N)** The intensity changes in cGMP fluorescence over time for the four conditions are shown. Gray traces show the optical responses to individual intensities (n > 6). The individual intensities are normalized (ΔF/F0). The dark line shows the mean value and the band the standard error range of the mean. The respective solution was acted on 2 min after the start of imaging (grey bars)."}

knitr::include_graphics(path = "figures/Fig4.png")

```

**Depolarisation of cPRC is required for activation of downstream interneurons and ciliomotor neurons**



therefore compared calcium imaging of postsynaptic cells (INNOS, INRGW, Ser-h1 and MC) in the cPRC circuit during UV stimulation of cPRC between WT and NOS mutant larvae.

We next quantified the responses of the INNOS and INRGW following a UV/violet stimulus to the cPRCs. To unambiguously identify the cells from which we recorded calcium signals, we developed an on-slide immunostaining method **(Figure 5---figure supplement 1)**. We used the cell-specific neuropeptide markers using anti-RYamide (INNOS) and anti-RGWamide (INRGW) antibodies [@Conzelmann2011] to mark the interneurons. Based on the position of the nuclei, we could correlate live and fixed samples at a single-cell precision **(Figure 5A,B)**. In both wild-type and *NOS*-mutant larvae, INNOS cells showed increase calcium signals during the inhibition phase of the cPRCs **(Figure 5D)**. In addition, INNOS depolarisation was suppressed in NIT-GC2 molphants where cPRC hyperpolarisation was suppressed **(Figure 5E)**. This indicates that cPRC hyperpolarisation is required for INNOS depolarisation. In contrast, calcium signals increased in INRGW cells in wild-type larvae during the second activation phase of cPRCs, a response that was lacking in *NOS*-mutants **(Figure 5F,G)**. This suggests that there is no circuit output to the Ser-h1 and MC cells and the ciliary band in *NOS*-mutants, consistent with the observed behavioural defects **(Figure 5H,I)**.


```{r echo=FALSE,fig.cap="**Figure 5. NO produced by UV stimulation to cPRCs induces a depolarising response in cPRC via NIT-GC1.** **(A, B)** Each left figure shows a calcium imaging using GCaMP6s. The respective right figures show immunostaining images of the same positions in the respective left figures, (A) shows RYamide antibodies, (B) shows RGWamide and serotonin (5-HT) antibodies in red and DAPI in cyan. Asterisks indicate the nuclei of the cPRC, respectively, and the nuclei of the cells that were identified from the arrangement of the surrounding cells are indicated by numbers. From the position of the cells, the calcium imaging data identify INNOS in (A) and INRGW and Ser-h1 in (B), respectively. **(C)** Correlation (Pearson's r) map of neuronal activity of the INNOS, INRGW, Ser-h1 and MC neurons. **(D)** Changes over time in GCaMP6s intensities in INNOS when cPRC is stimulated with UV in WT (dark purple) and NOSΔ11/Δ11 mutant (light purple) larvae. **(E)** Changes over time in GCaMP6s intensities in INNOS when cPRC is stimulated with UV in NIT-GC2 morphants (MO1 and MO2: pink and light pink) larvae. **(F)** Changes over time in GCaMP6s intensities in INRGW when cPRC is stimulated with UV in WT (dark blue) and NOSΔ23/Δ23 mutant (light blue) larvae. **(G)** Changes over time in GCaMP6s intensities in INRGW when cPRC is stimulated with UV in NIT-GC2 morphants (MO1 and MO2: blue and light blue). **(H, I)** Changes over time in GCaMP6s intensities in Ser-h1 and MC when cPRC is stimulated with UV in WT (Ser-h1 and MC: dark red and brown) and NOSΔ11/Δ11 mutant (Ser-h1 and MC: light red and brown) larvae."}

knitr::include_graphics(path = "figures/Fig5.png")

```


## Discussion

In the present study, we explored the role of NO in the UV avoidance behaviour of Platynereis. We demonstrated that there are interneurons expressing NOS in the cPRC circuit and that NO is required for UV avoidance in behavioural experiments using NOS mutants. We identified the axon-localised type NIT-GC1 and the cilia-localised type NIT-GC2 expressed in the cPRCs as targets of NO and showed from cultured cell experiments that these produce cGMP in response to NO. We have shown by calcium imaging that INNOSs are activated in response to the hyperpolarising response of the cPRCs, that the NO produced feeds back to the cPRC via NIT-GC1 and induces a depolarising response in the cPRC, and that the depolarising response of the cPRC is involved in activating downstream INRGW. Lastly, we confirmed that RGWamide induces the downward swimming behaviour of larvae.


```{r echo=FALSE,fig.cap="**Figure 6. Gene expressions and signalling mechanisms between cPRC, INNOS and INRGWa.** **(C)** Dot plot of genes (columns) expressed in three types of cells (rows) in the cPRC circuit using single cell RNA-Seq. The size of the dots is expressed in proportion to the percentage of cells expressing that gene relative to all cells. The colours represent the normal logarithm of the number of transcripts in the cells expressing the gene. **(D)** Schematic diagram of the signalling pathway of the cPRC circuit, focusing on the NO feedback."}

knitr::include_graphics(path = "figures/Fig6.png")

```


cPRCs expressing UV-absorbing c-opsin1 have high resting calcium levels and show hyperpolarising and subsequent depolarising responses to UV light (405 nm) [@Veraszto2018]. In the present study, we have shown that the cells that strongly synapse with cPRCs are interneurons that specifically express NOS (INNOS). The fluorescence intensity of DAF-FM, which specifically detected INNOS, showed a transient tonic-like increase dependent on UV stimulation of the cPRCs. The graph shows a decrease in intensity after UV light is switched off, but the response of DAF-FM to NO is irreversible and this decrease is due to breaching by the relatively strong excitation light. Attempts to measure changes in the intensity of the calcium indicator RGECO1a simultaneously with DAF-FM imaging did not lead to detection of changes in fluorescence intensity compared to GCaMP6s and could only visualise the cPRCs. In calcium imaging using GCaMP6s, INNOS identified by the positional relationship between the cPRC and INNOS and the labelling of RYa depolarised in clear synchrony with the hyperpolarising response of the cPRCs. This tonic response may involve dopaminergic nerves based on single-cell data **(Figure 6)** [@puopolo2001][@zhang2007][@Randel2014a]. In the mammalian retina, it is known that signals from bipolar photoreceptor cells are transmitted to downstream NOS-expressing amacrine cells (NOAC), where NO is produced [@jacoby2018]. These suggest that UV stimulation of cPRCs causes the production of NO via a depolarising response of INNOS.

Our results suggest that NIT-GC1, which is specifically expressed in cPRCs, functions as a target for NO produced by INNOS and contributes to the regulation of the cPRC circuit transmission pathway via a depolarising response by increasing cGMP. In the mammalian retina, NO is an essential signalling molecule and one of the most well-studied [@Cudeiro1999]. It has been reported that NO produced in one of the retinal cells modulates cGMP-mediated neurotransmission by activating presynaptic neural sGCs (general target of NO) through retrograde signalling [@Vielma2014][@Wei2012]. In Platynereis cPRC, NIT-GC1 was also identified as having the property of producing cGMP in a NO-dependent manner. Furthermore, it is known that CNGAα, one of the cyclic nucleotide-gated channels, is specifically highly expressed in cPRC [@Tosches2014], and in INNOS-cPRC signalling, NO is implicated in the neuromodulation of the cPRC by retrograde signalling. NO is a molecule with a very short half-life, and when functioning as a signalling molecule, NOS and sGC are known to localise very close together at synapses between neurons [@burette2002][@garthwaite2015a]. In our study, we found 12 sGCs in Platynereis, but none expressed in the cPRC. Localisation analysis of NOS and NIT-GC1 by antibody staining showed that the two proteins localised very close together in the anterior neurosecretory plexus region. Calcium imaging results using NOS mutant and NIT-GC1 knockdown larvae suggest that NIT-GC1 activation induces a depolarising response in the cPRC.

Another type of interneuron that strongly synapses with the cPRC is the INRGWa, which synapses directly onto the ciliomotor neuron and is therefore considered to be a very important neuron in the control of larval behaviour [@Veraszto2018][@williams2017]. The present results suggest that the depolarising response of the cPRC due to NO feedback induces INRGW activation. RGWamide is also a neuropeptide that induces larval downward swimming quite strongly (unpublished? need to show data?), and the delayed downward swimming of NOS mutant larvae in response to UV stimulation may be responsible for the failure of INRGW activation to occur. In addition, in many marine invertebrates, NO has been reported to be involved in many aspects of settlement behaviour and metamorphosis induction [@leise2001; @bishop2003; @bishop2007; @pechenik2007; @castellano2014; @ueda2013; @ueda2014; @ueda2016; @yang2018; @zhu2020; @song2021]. The downward swimming behaviour of Platynereis larvae is the first step in the onset of settlement behaviour and metamorphosis. These suggest that downward swimming due to NO activation of INRGWs may trigger the onset of settlement behaviour and metamorphosis [@Conzelmann2013][@Williams2015]. These suggest that NO feedback signalling may act as a trigger for downward swimming via activation of INRGWs, which induces the initiation of settlement behaviour and metamorphosis.

An outstanding question is how the NO\~NIT-GC response alters neuronal dynamics. NO is a single molecule and, due to its short half-life, cannot be transmitted over long distances and has a limited signalling repertoire. It has therefore been considered unsuitable for generating diversity in signalling compared to other signalling molecules such as neuropeptides and classical neurotransmitters [@Jékely2021]. However, our results in the present study indicate that NO-induced activation of NIT-GC1 is involved in the regulation of neuronal circuits (cPRC pathway). The NIT domain was originally found in bacteria and the existence of GCs with NIT domains was first reported by Moroz et al in 2020 [@Shu2003; @Moroz2020]. It has been proposed that NIT domains in bacteria regulate cellular functions in response to changes in extracellular and intracellular nitrate and/or nitrite concentrations [@camargo2007]. In fact, NO is a free radical molecule, with a half-life of 5 msec in vivo, and is readily converted to nitrate and nitrite [@santos2011; @möller2019; @garthwaite2015a]. And nitrate and nitrite have been reported to accumulate in cells and tissues with high NOS activity in placozoans and cnidarians [@Moroz2004; @Moroz2020]. It is possible that the NIT domains of NIT-GC1 identified in this study may also be used for nitrate/nitrite sensing, as in bacteria. Furthermore, if they show different sensitivities to NO, nitrite and nitrate, it is possible that the temporal variation in their respective half-lives could give rise to a repertoire of activation timings [@lundberg2011]. In addition, we have found at least 15 NIT-GCs in the Platynereis transcriptome data, of which NIT-GC2 is highly expressed in the cPRC, in addition to NIT-GC1. Interestingly, NIT-GC2 showed a very different localisation pattern to NIT-GC1, and furthermore, the results of calcium imaging of cPRC in response to UV stimulation using NIT-GC2 knockdown larvae were the different dynamics (such that hyperpolarisation was suppressed) with NIT-GC1 knockdown. It was not clear from the present analysis whether this difference was due to differences in the localisation or the property of the NIT-GCs. However, the diversification and spatially different localisation patterns of NIT-GCs may increase their signalling repertoire. We also believe that the appearance of NIT-GCs was one of the means to generate the diversity of signalling pathways using NO during the early stages of nervous system evolution in metazoans, since sponges and ctenophores, where NO signalling is present, have no NIT-GCs at all, as many as 12 have been found in placozoans, and NIT-GCs are functional in the "proto-eye" circuit of Platynereis.

## Materials and Methods

**CRISPR-Cas9 Design and Microinjection**

Before designing the small guide RNA (sgRNA) for the sgRNA:Cas9 nuclease, splice sites and polymorphic sites in our laboratory culture were identified to avoid them. The sgRNA targeted the third exon of Platynereis dumerilii NOS (target site: 5'-GGGCAATACTGGCTCCACTC-3'). The sgRNA was assembled from two annealed oligonucleotides (5'-TAGGGCAATACTGGCTCCACTC-3', 5'-AAACGAGTGGAGCCAGTATTGC-3') forming overhangs for cloning into a BsaI site of the plasmid pDR27456 (Hwang et al. 2013)(42250, Addgene), which contains next to the BsaI site a tracrRNA sequence. The plasmid was then used to PCR amplify DNA (primers: T7, 5'-AAAAGCACCGACTCGGTGCC-3') for synthesizing the sgRNA. The DNA was purified with the QIAquick PCR Purification Kit (Qiagen). From the DNA, the sgRNA was synthesized with the MEGAshortscript Kit (Thermo Fisher Scientific) and was purified with the MEGAclear Kit (Thermo Fisher Scientific). Cas9-mRNA was transcribed, capped, and polyA-tailed with the mMessage mMachine Kit and the Poly(A) Tailing Kit (both Thermo Fisher Scientific) from a plasmid (pUC57-T7-RPP2-Cas9) containing the Cas9 ORF fused to 169 base pair 5' UTR from the Platynereis dumerilii 60S acidic ribosomal protein P2. The sgRNA (18 ng/ml) and the Cas9-mRNA (180 ng/µl) were coinjected into fertilized eggs of Platynereis dumerilii wild-type parents according to an established injection procedure [@Conzelmann2013a]. The eggs were kept at 18°C for 45 min before injection and were injected at 14.5°C. The injected individuals were kept at 18°C for 5 to 8 days in 6-well- plates (Nunc multidish no. 150239, Thermo Scientific) and then cultured at 22°C until sexual maturity. The mature worms were crossed to wild-type worms and the progeny was genotyped, resulting in two founder lines, which were bred to homozygosity.

**NOS sequencing and genotyping**

For genotyping of the NOS locus, genomic DNA was isolated from single larvae, groups of 6-20 larvae, or from the tails of adult worms. The DNA was amplified by PCR (primers: 5'-GGTTCATTGGTTTCGATAACATTGCGG-3', 5'-CAGAGTCGATCAGTCTGCATATCTCCA-3') with the dilution protocol of the Phusion Human Specimen Direct PCR Kit (Thermo Scientific). The PCR product was sequenced directly with a nested sequencing primer (5'-GGTGCTCTCCCGGGTACACAA-3'). A mixture of wild-type and deletion alleles in a sample gave double peaks in the sequencing chromatograms, with the relative height of the double peaks reflecting the relative allele ratio in the sample.

**Vertical column setup for measuring photoresponses**

Photoresponses of larvae of different ages were assayed in a vertical Plexiglas column (31 mm x 10 mm x 160 mm water height). The column was illuminated from top with light from a monochromator (Polychrome II, Till Photonics). The monochromator was controlled by AxioVision 4.8.2.0 (Carl Zeiss MicroImaging GmbH) via analog voltage. The light passed a collimator lens (LAG-65.0-53.0-C with MgF2 Coating, CVI Melles Griot) before entering the column. The column was illuminated from both sides with light-emitting diodes (LEDs). The LEDs on each side were grouped into two strips. One strip contained UV (395 nm) LEDs (SMB1W-395, Roithner Lasertechnik) and the other infrared (810 nm) LEDs (SMB1W-810NR-I, Roithner Lasertechnik). The UV LEDs were run at 4 V to stimulate the larvae in the column from the side. The infrared LEDs were run at 8 V (overvoltage) to illuminate the larvae for the camera (DMK 22BUC03, The Imaging Source), which recorded videos at 15 frames per second and was controlled by IC Capture (The Imaging Source).

**Comparing behavior of wildtype and NOS-knockout 3-day-old larvae**

To compare the behavior of wildtype and NOS-knockout larvae at 3 days in the vertical column, the larvae were mixed and left in the dark for 5 min. The larvae were treated with NOS inhibitors for pharmacology. The NOS inhibitors were L-NAME. The larvae were treated with different concentrations in adjacent columns. The concentrations for the NOS inhibitors were control, 1 mM, 0.1 mM. The larvae were recorded for 1 min in the dark followed by exposure to collimated cyan (480 nm) light from the top of the column for 2 min, then 2 min darkness, and finally collimated UV (395 nm) light from the top of the column for 2 min. Stimulus light was provided by the monochromator (Polychrome II, Till Photonics). Scripts are available at <https://github.com/JekelyLab/NOS>.

**NOS Identification and Phylogenetic Analysis**

To identify NOS, we obtained a "seed" database of oxygenase domain in Pfam database, PF02898. From these sequences, we produced a Hidden Markov Model (HMM) and used this to mine the 47 metazoan species, 2 choanoflagellate species and 2 filasterea species investigated. HMM models were run in HMMR3 with an e-value of 1e−15. We ran CD-Hit [@Fu2012] to eliminate redundant sequences (at a 80% threshold). We aligned the sequences with MAFFT version 7, with the iterative refinement method E-INS-i. Alignments were trimmed with TrimAl in gappy-out mode [@Capella-Gutiérrez2009]. To calculate maximum-likelihood trees, we used IQ-tree2 with the LG+G4 model. To calculate branch support, we ran 1,000 replicates with the aLRT-SH-like and aBayes methods [@Minh2020]. The sequences used for the phylogenetic analysis are available in **Supplementary File 2**, the trimmed alignment is available in **Supplementary File 3**.

**NIT-GC Identification and Phylogenetic Analysis**

To identify NIT-GCs, we obtained a "seed" database of Adenylate and Guanylate cyclase catalytic domain in Pfam database, PF00211. From these sequences, we produced a Hidden Markov Model (HMM) and used this to mine the 45 metazoan species, 2 choanoflagellate species and 2 filasterea species investigated. HMM models were run in HMMR3 with an e-value of 1e−15. We ran CD-Hit [@Fu2012] to eliminate redundant sequences (at a 80% threshold). The CLANS analysis is available as **Supplementary File 1**. To identify clusters, we used the convex-clustering option with 100 jack-knife replicates. The NIT-GCs are extremely well conserved in membrane-bound guanylate cyclases and form an easily recognizable cluster. To analyze the phylogeny of NIT-GCs, the cluster containing these GCs together with membrane-bound guanylate cyclases were parsed and used for tree building. We aligned the sequences with MAFFT version 7, with the iterative refinement method E-INS-i. Alignments were trimmed with TrimAl in gappy-out mode [@Capella-Gutiérrez2009]. To calculate maximum-likelihood trees, we used IQ-tree2 with the LG+G4 model. To calculate branch support, we ran 1,000 replicates with the aLRT-SH-like and aBayes methods [@Minh2020]. The sequences used for the phylogenetic analysis are available in **Supplementary File 2**, the trimmed alignment is available in **Supplementary File 3**.

**Single-cell analysis**

We used Achim et al. for the single-cell data [@Achim2015]. In fact, in Williams et al., they used 107 cells as neurons by removing duplicates from Achim et al. single-cell data, so we used those cells [@williams2017]. Since the raw data were read count data, we normalized them to TPM using Python. After that we converted them to log10. From the sum of the expression levels in 107 cells for each gene, I calculated the percentage in each cell. We chose the appropriate genes for now. For each cell, we identified them with marker genes. After created the data in Python, plotted it using R dot plots. RPKM calculates the total number of reads per million bp and then divides by the length of each gene, so it is not possible to compare between samples. Instead, TPM first divides by the length of the gene and then divides by the total number of reads per million bases, which allows for more accurate comparisons between samples. In this case we wanted to compare between samples, so we used TPM. In fact, TPM is becoming the mainstream method for many single-cell analyses. The total TPM of each gene between the samples was used to calculate the percentage of expressed genes.

**In situ HCR**

Larvae were fixed and treated with Proteinase K, according to the conventional WMISH protocol [@Tessmar-Raible2005], with fixation in 4% paraformaldehyde/ PTW (PBS with 0.05% Tween20) for 2 hr at room temperature, and Proteinase K treatment in 100 µg/ml Proteinase K/ PTW for 3 min [@Tessmar-Raible2005]. Specifically, for the HCR protocol, samples were processed in 1.5 ml tubes. Probe hybridization buffer, probe wash buffer, amplification buffer, and fluorescent HCR hairpins were purchased from Molecular Instruments (Los Angeles, USA). Hairpins associated with the b2 initiator sequence were labeled with Alexa Fluor 647, and the hairpins associated with the b3 initiator sequence were labeled with Alexa Fluor 546. To design probes for HCR, we used custom software [@Kuehn2022] to create 20 DNA oligo probe pairs specific to P. dumerilii NOS, NIT-GC1, NIT-GC2, RYa-pNP (GenBank accession: JF811330.1), c-opsin1 (GenBank accession: AY692353.1), CNGAα (GenBank accession: KM199644.1), and MLD/pedal2-pNP (GenBank accession: KF515945.1). The NOS, NIT-GC1 and NIT-GC2 probes were designed to be associated with the b2 initiator sequence, while the RYa-pNP, c-opsin1, CNGAα and MLD/pedal2-pNP probes were designed to be associated with the b3 initiator sequence. For the detection stage, samples were pre-hybridized in 200 µl of probe hybridization buffer for 1 hr at 37°C, and then incubated in 250 µl hybridization buffer containing probe oligos (4 pmol/ml) overnight at 37°C. To remove excess probe, samples were washed 4× with 1 ml hybridization wash buffer for 15 min at 37°C, and subsequently 2× in 1 ml 5× SSCT (5× SSC with 0.1% Tween20) for 5 min at room temperature. For the amplification stage, samples were pre-incubated with 100 µl of amplification buffer for 30 min, room temperature, and then incubated with 150 µl amplification buffer containing fluorescently labeled hairpins (40nM concentration (2ul of 3uM stock in 150ul amplification buffer, snap-cooled as described; [@Choi2018]) overnight in the dark at 25°C. To remove excess hairpins, samples were washed in 1 ml 5× SSCT at room temperature, twice for 5 min, twice for 30 min, and once for 5 min. During the first 30 min wash, samples were counterstained with DAPI (Cat. #40043, Biotium, USA).

**Immunohistochemistry**

Whole-mount immunostaining of 2 day old Platynereis larvae fixed with 4% paraformaldehyde were carried out using primary antibodies raised against NIT-GC1, NIT-GC2, NOS, RYamide neuropeptide, RGWamide neuropeptide in rabbit, plus a commercial antibody raised against acetylated tubulin in mouse (Sigma T7451). The synthetic peptides contained an N-terminal Cys that was used for coupling during purification. Antibodies were affinity purified from sera as previously described [@Conzelmann2012]. Immunostainings were carried out as previously described [@Conzelmann2012]. The NOS promoter (fragment sizes: 12 Kb) was amplified and cloned upstream of 3xHA- Palmi-tdTomato. Larvae injected with promoter constructs (ca. 250 ng/ml) were analysed for reporter expression at 3 days post fertilization using an AxioImager Z.1 fluorescence wide-field microscope (Carl Zeiss GmbH, Jena) and immediately fixed for immunostainings. The protocol followed for immunostaining of HA-tagged reporters was recently described [@Veraszto2017]. Specimens were imaged with a LSM 780 NLO or LSM 880 with Airysan Confocal Microscope (Zeiss, Jena).

**Calcium imaging**

For calcium imaging, 49--55 hpf larvae were used. Experiments were performed at room temperature and larvae were immobilised by being embedded in 2.5% agarose filtered artificial seawater between a slide and coverslip spaced with adhesive tape. GCaMP6s mRNA (1 mg/ml) was injected into zygotes as described previously [@Randel2014a]. Larvae were imaged on a Zeiss LSM 880 with Airyscan (with a C-Apochromat 63X/1.2 Corr - water) with a frame rate of 1.88 frame/sec and an image size of 512 x 512 pixels. The larvae were stimulated in a region of interest (a circle with ?? pixel diameter) with 405 nm lasers controlled by the Bleaching mode. The imaging laser had a similar intensity than the stimulus laser but covered an area that was 10 times larger than the stimulus ROI.

**Cell culture experiment**

Green cGull was used for the cGMP assay [@Matsuda2017]. A full-length Pdum-NIT-GC1 and -NIT-GC2 coding sequences were amplified by PCR starting from a *Platynereis dumerilii* cDNA library and cloned into the pcDNA3.1(+) vector using the T2A self-cleaving sequence. Cos-7 cells with low expression of endogenous soluble guanylate cyclase were used as cultured cells for gene expression. This cell line was purchased from Angio-proteomie (CAT no. cAP-0203). The Cos-7 cells were maintained at 37 °C in 35mm dishes (Nunc™ Glass Bottom Dishes) containing 3 mL of DMEM, high glucose glutamax medium (Thermo; Cat. No. 10566016) supplemented with 10% fetal bovine serum (Thermo; Cat. No. 10082147). Upon reaching confluency of approximately 85%, we transfected the cells with the plasmid containing Green cGull-T2A-NITGC1. Transfections were carried out with 150 ng of each plasmid and 0.3 μl of the transfection Lipofectamine 3000 Reagent (invitrogen; Cat. No. L3000001). Two days post-transfection, we removed the culture medium and substituted it for fresh DMEM-medium. For single-wavelength imaging experiments, cells in 35-mm dishes were washed twice and imaged in modified Ringer's buffer (140 mM NaCl, 3.5 mM KCl, 0.5mM NaH2PO4 , 0.5mM S-3 MgSO4 , 1.5 mM CaCl2 , 10 mM HEPES, 2 mM NaHCO3 and 5 mM glucose). Dishes were mounted on a stage heated at 37 °C and imaging was performed using an inverted microscope (LSM880, Zeiss) equipped with an oil-immersion objective lens (UApo/340, 40, NA = xx). Images were acquired using a xenon lamp, 460--495 nm excitation filter, 505-nm dichroic mirror and 510-- 550-nm emission filter (????). S-Nitroso-N-acetyl-D, L-penicillamine (SNAP) was purchased from Sigma-Aldrich (St. Louis, MO, USA) . The exposure time of the EM-CCD camera was controlled by the ZEN software (Zeiss). Images were acquired every 15 s for 10 min and stimulation was initiated 2 min after starting image acquisition. Imaging data analysis was performed using ImageJ (National Institutes of Health, Bethesda, MD, USA).

## Acknowledgements

This work was funded by the Wellcome Trust (214337/Z/18/Z).

## References {.unnumbered}

::: {#refs}
:::


## Figure supplements


```{r echo=FALSE,fig.cap="**Figure 1 -- figure supplement 1** "}
knitr::include_graphics(path = "figures/Fig1_sup1.png")

```

```{r echo=FALSE,fig.cap="**Figure 1 -- figure supplement 2** "}
knitr::include_graphics(path = "figures/Fig1_sup2.png")

```

```{r echo=FALSE,fig.cap="**Figure 3 -- figure supplement 1** "}
knitr::include_graphics(path = "figures/Fig3_sup1.png")

```

```{r echo=FALSE,fig.cap="**Figure 4 -- figure supplement 1** "}
knitr::include_graphics(path = "figures/Fig4_sup1.png")

```

```{r echo=FALSE,fig.cap="**Figure 4 -- figure supplement 2** "}
knitr::include_graphics(path = "figures/Fig4_sup2.png")

```

```{r echo=FALSE,fig.cap="**Figure 5 -- figure supplement 1** "}
knitr::include_graphics(path = "figures/Fig5_sup1.png")

```




