---
title: Nitric oxide feedback to ciliary photoreceptor cells orchestrates UV avoidance
  in zooplankton
author: 
  - name: "Kei Jokura<sup>1,`*`</sup>, Nobuo Ueda<sup>2</sup>, Martin Gühmann<sup>3</sup>, Luis Alfonso Yañez-Guerra<sup>1</sup>, Piotr Słowiński<sup>1</sup>, Kyle C. A. Wedgewood<sup>1</sup> and Gáspár Jékely<sup>1,4,`*`</sup>"
    affiliation: "<sup>1</sup>Living Systems Institute, University of Exeter, Stocker road, Exeter, EX4 4QD, United Kingdom <sup>2</sup>Okinawa Institute of Science and Technology, Okinawa, Japan, <sup>3</sup>School of Biological Sciences, University of Bristol, Bristol, United Kingdom, <sup>4</sup>Centre for Organismal Studies (COS), University of Heidelberg, 69120 Heidelberg, Germany  <br> <sup>*</sup>Correspondence`:` k.jokura @ exeter.ac.uk, gaspar.jekely @ cos.uni-heidelberg.de"
output:
  html_document:
    theme: lumen
  pdf_document:
    latex_engine: lualatex
    template: default
  word_document: default
geometry: left=0.8in,right=0.8in,top=0.8in,bottom=0.5in
mainfont: Arial
monofont: Times New Roman
documentclass: article
classoption: onecolumn
linestretch: 1.2
fontsize: 10pt
bibliography: references.bib
csl: elife.csl
editor_options: 
  markdown: 
    wrap: sentence
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)

```

## Abstract

Nitric oxide (NO) generated by nitric-oxide synthase (NOS) is a key regulator of animal physiology.
Here we uncover a role for NO in shaping circuit dynamics to orchestrate light-avoidance behaviour.
We studied UV/violet light avoidance mediated by brain ciliary photoreceptors (cPRCs) in larval *Platynereis dumerilii*, a marine annelid.
We found NOS expressed in interneurons (INNOS) postsynaptic to cPRCs.
Stimulation of cPRCs by violet light leads to cPRC inhibition but concomitant INNOS activation and NO production.
NO feeds back to cPRCs and triggers their delayed activation through an unconventional guanylyl cyclase.
This results in the activation of projection interneurons and the inhibition of serotonergic ciliomotor neurons.
In *NOS* mutants, NO feedback and projection-neuron activation do not occur and avoidance behaviour is defective.
By mathematical modelling, we could recapitulate phototransduction and circuit dynamics in both wild-type and mutant larvae.
Our results reveal how NO-mediated retrograde signalling gates a synaptic circuit to initiate light-avoidance behaviour.

## Introduction

In nervous systems, synaptic transmission and volume transmission together shape circuit dynamics.
While synaptic transmission occurs at specialised contact sites, volume transmission is characterised by the delocalised release of diverse diffusive neuromodulators.

Nitric oxide (NO) is one such modulator with unique physical and signalling properties.
This free radical synthesized by nitric oxide synthase (NOS) from L-arginine is short-lived and can diffuse across biological membranes [@Cudeiro1999; @thomas2015].
Canonical NO signalling involves the Ca2+/calmodulin-dependent activation of NOS following calcium increase, NO production and diffusion, and the NO-dependent activation of soluble guanylate cyclases (sGC) cell-autonomously or in other cells leading to cGMP production [@Bredt1990; @Hoelscher1997].
Given that NOS activation is calcium dependent and NOS shows neuron-type-specific expression [@Gibbs1998a; @Wildemann1999; @Aso2019; @Mobley2022], NO action can lead to the activity-dependent modulation of neural circuits at specific sites [@Wang2007; @Vielma2014; @jacoby2018; @Aso2019] .

NO signalling has been extensively studied in the vertebrate retina where NOS is expressed in amacrine, ganglion as well as other cells [@Cudeiro1999; @Wang2007; @jacoby2018].
The actions of NO can be diverse and not always involve the canonical sGC-cGMP pathway [@Tooker2013; @Wei2012; @jacoby2018].
Defective NO signalling in NOS knockouts for example leads to a decreased sensitivity of retinal ganglion cells to light stimulation [@Wang2007].
However, due to the complex expression of NOS in vertebrates and the diversity of its functions, it has been challenging to link the neurophysiological effects of NO to signalling mechanisms and behaviour change.

In the whole organism context, NO signalling has often been studied in diverse marine invertebrates, where NO can regulate larval settlement and metamorphosis [@leise2001a; @ueda2016; @Locascio2022; @song2021; @Zhang2012].
While NOS-expressing neurons potentially responsible for these effects have been reported [@bishop2007; @Locascio2022], it has not been possible to link these NO-dependent effects on behaviour or life-cycle transitions to neuronal activity and function.

Due to these difficulties, we still know little at the whole organism level about how NO production relates to stimulus conditions, how it shapes circuit activity at specific neuron types, and how NO-dependent modulation relates to behaviour.

To investigate NO function in neural circuit dynamics and behaviour, here we study larvae of the marine annelid model *Platynereis dumerilii* [@ozpolat2021].
*Platynereis* has emerged as a systems neuroscience model where behaviour and activity imaging can be combined with genetic manipulations and a whole-body synaptic connectome and gene expression atlases are available [@Vergara2021; @Veraszto2017; @veraszto2020; @ozpolat2021].

Here we uncovered an essential function for NO signalling in larval UV/violet-light avoidance behaviour.
In *Platynereis* larvae, UV/violet avoidance is mediated by brain ciliary photoreceptor cells (cPRCs) and is characterised by downward swimming [@Veraszto2018].
The cPRCs express a ciliary-type opsin, c-opsin1 [@Arendt2004] that forms a UV-absorbing bistable photopigment with an absorption maximum around 384 nm [@tsukamoto2017, @Veraszto2018, @Veedin2021].
Upon UV/violet exposure, the cPRC show a characteristic biphasic calcium response that is *c-opsin1* dependent.
UV/violet avoidance is also *c-opsin1*-dependent and is defective in *c-opsin1* mutants [@Veraszto2018].
Here we show that *Platynereis* *NOS* is expressed in interneurons of the cPRC circuit and is required for UV/violet-avoidance.
By combining calcium imaging across the fully-mapped cPRC circuit [@Veraszto2018] with genetic perturbations and mathematical modelling, we describe how NO tunes circuit dynamics through non-synaptic retrograde signalling to cPRCs.
This delayed neuroendocrine feedback integrates UV/violet exposure to induce a short-term memory manifested in altered circuit activity and an aversive behavioural response.

## Results

**Nitric oxide synthase is expressed in interneurons of the UV-avoidance circuit**

We identified a single *nitric oxide synthase* (*NOS*) gene in the *Platynereis dumerilii* genome and transcriptome data.
Phylogenetic analysis of NOS proteins indicate that *Platynereis* NOS belongs to an orthology group of metazoan NOS sequences (Figure 1---figure supplement 1).
To characterise the expression pattern of *NOS* we used in situ hybridization chain reaction (HCR) and transient transgenesis.
In two- and three-day-old larvae, we detected *NOS* expression in four cells (two of them weakly expressing) in the apical organ region (Figure 1D and Figure 1---figure supplement 2).
*NOS* was also expressed in the region of the visual eyes (adult eyes) and the pigmented eyespots (Figure 1---figure supplement 2).
The four apical organ cells, but not the eyes, were also labelled with a *NOS* reporter construct driving palmitoylated-tdTomato (Figure 1E).
This reporter also revealed the axonal projections of these central NOS-expressing neurons.
The position and morphology of the four *NOS+* cells allowed us to identify the same four cells as four interneurons (INNOS) in our three-day-old whole-body *Platynereis* volume EM data [@williams2017; @veraszto2020] (Figure 1B,C).
In the synaptic connectome, the INNOS cells are postsynaptic to the UV-sensory cPRCs and presynaptic to the INRGW interneurons, which are also cPRC targets (Figure 1C,F).
The projections of INNOS cells are segregated into input and output compartments, with cPRCs inputs occurring in the dense neurosecretory plexus.
INNOS output synapses form in the more ventral projection region (Figure 1---figure supplement 3).
INRGW neurons synapse on the head serotonergic ciliomotor neurons (Ser-h1), which synapse on the prototroch ciliary band and the cholinergic MC ciliomotor neurons (Figure 1C,F) [@Veraszto2017].

```{r echo=FALSE,fig.cap="**Figure 1. Identification of NOS-expressing interneurons (INNOS) within the cPRC circuit.** **(A)** Scanning electron microscopy image of a three-day-old *Platynereis* larva. **(B, C)** Volume rendering of the neuron types (cPRC, INNOS, INRGW, Ser-h1 and MC) in the cPRC circuit reconstructed from a whole-body transmission electron microscopy volume of a three-day-old larva. Neurite skeletons are shown with cell-body positions represented by spheres. Projections of all neurons in the body are shown in grey to highlight the neuropils. The outline of the yolk is also indicated in grey. In B, nuclei positions of the prototroch head ciliary band are shown as grey spheres. **(D)** Expression of the *NOS* gene detected by in situ HCR (magenta) in a two-day-old larva (anterior view). Antibody staining for acetylated α-tubulin (acTub: green) highlights cPRC cilia and the neuropil. **(E)** Expression of a *NOS* reporter (NOSp::palmi-3xHA: magenta) labelled with an anti-HA antibody in a two-day-old larva (anterior view). Antibody staining for acetylated α-tubulin (acTub: green) highlights cPRC cilia and the neuropil. **(F)** Synaptic wiring diagram of the cPRC circuit. Hexagons represent cell groups, with the number of cells per group shown in square brackets. Arrows represent the summed number of synaptic contacts between cell groups. Arrow thickness is proportional to the number of synapses."}

knitr::include_graphics(path = "figures/Fig1.png")

```

**Nitric oxide is produced during UV/violet stimulation of the cPRCs**

The expression of *NOS* in the INNOS interneurons in the cPRC circuit suggests that NO signalling may be involved in UV/violet-avoidance.
To test this, first we asked whether NO is produced during UV/violet stimulation of the larvae.
We injected the fluorescent NO-reporter DAF-FM into zygotes and imaged two-day-old larvae while exposing the region of cPRC cilia to 405 nm violet light.
To mark cell outlines, we coinjected mRNA encoding a red-fluorescent reporter (RGECO), allowing the identification of the cPRCs.
Following light stimulation in the region of the ramified cilia of the cPRCs, we detected an increase in DAF-FM fluorescence in the anterior neurosecretory neuropil, the region of INNOS projections.
This increase did not occur in larvae where we illuminated a control area (Figure 2).

```{r echo=FALSE,fig.cap="**Figure 2. NO produced by UV/violet stimulation to cPRCs.** **(A)** DAF-FM fluorescence in the region of the neurosecretory neuropil. The white line indicates the outline of the larva. The dashed line corresponds to the area where fluorescence was quantified. Circles indicate the location of cPRC and control stimulation. cPRCs are marked by thin lines. **(B)** Changes in DAF-FM fluorescence before and during 405 nm light stimulation. **(C)** Changes in DAF-FM fluorescence over time during 405 nm stimulation of the cPRCs (green) or a control area (ctr: gray). The purple box indicate the duration of 405 nm stimulation. Individual traces normalized (ΔF/F0) are shown as thin lines. Thick lines show the mean value with 0.95 confidence intervals. N = 9 larvae for control and 11 for cPRC stimulation."}

knitr::include_graphics(path = "figures/Fig2.png")

```

**Nitric oxide signalling mediates UV-avoidance behaviour**

We next tested whether NO signalling is required for UV/violet avoidance.
To achieve this, we generated two *Platynereis* *NOS* knockout lines with the CRISPR/Cas9 method.
We recovered two deletions (NOSΔ11/Δ11 and Δ23/Δ23), both leading to frame-shift and an early stop codon and thus likely representing null alleles (Figure 3---figure supplement 1A).
We could establish a homozygous line for both mutations indicating that NOS is not an essential gene in *Platynereis*.
To quantify UV avoidance, we recorded the trajectories of freely swimming wild type and mutant larvae in vertical columns, illuminated laterally from two opposite sides with 395 nm UV light (Figure 3A and Figure 3---figure supplement 1B).
As previously shown, wild-type larvae swim downward following non-directional UV/violet light stimulation [@Veraszto2018].
In contrast, both two- and three-day-old homozygous *NOS*-mutant larvae showed a strongly diminished UV-avoidance response (Figure 3A, B and Figure 3---figure supplement 1B,C).
This phenotype is similar to the defective UV-avoidance of *c-opsin1* mutant larvae and reveals a requirement for NOS in UV-avoidance behaviour [@Veraszto2018].
Wild type but not mutant larvae also showed an increase in swimming speed under UV light that may be due to downward swimming trajectories (swimming in the direction of gravity) (Figure 3B and Figure 3---figure supplement 1C).
We also tested directional phototaxis, by exposing larvae to 480 nm directional collimated light from the top of the column.
Three-day-old but not two-day-old *NOS*-mutant larvae also showed reduced phototactic behaviour, suggesting a function for *NOS* in the visual eyes that mediate three-day-old phototaxis [@Randel2014a] (Figure 3D and Figure 3---figure supplement 1G).
To distinguish between an acute and developmental function of NOS in light responses, we next tested larvae exposed to the NOS inhibitor L-NAME.
Larvae incubated for 5 min in 0.1 mM or 1 mM L-NAME showed a dose-dependent inhibition of UV avoidance.
In contrast, phototaxis was not affected (Figure 3C, E).
Overall, our results indicate an acute requirement for NOS signalling in UV-avoidance and a possible indirect, developmental role in the visual system, reminiscent of the function of NO signalling in *Drosophila* eye development [@Gibbs1998a].

```{r echo=FALSE,fig.cap="**Figure 3. NOS is required for UV avoidance in *Platynereis* larvae.**  **(B)** Swimming trajectories of wild type (WT, n=32) and NOS mutant (NOSΔ11/Δ11, n=26 and NOSΔ23/Δ23, n=47) three-day-old larvae. All trajectories start at 0 x and y position and time 0 corresponding to 10 sec after the onset of 395 nm stimulation from the side. **(B)** Vertical position of batches of wild type and mutant three-day-old larvae over time under 395 nm UV stimulation. The starting position of each larval trajectory was set to 0. **(C)** Vertical position of batches of control and L-NAME-treated (0.1 and 1 mM) three-day-old larvae over time under 395 nm UV stimulation. The starting position of each larval trajectory was set to 0. **(D)** Vertical displacement in 30 sec bins of wild type and mutant (NOSΔ11 and NOSΔ23) three-day-old larvae stimulated with 395 nm light from side, 488 nm light from the top and 395 nm light from the top. **(E)** Vertical displacement in 30 sec bins of control and L-NAME-treated (0.1 and 1 mM) three-day-old larvae stimulated with 395 nm light from the side, 488 nm light from the top and 395 nm light from the top."}

knitr::include_graphics(path = "figures/Fig3.png")

```

**NO retrograde signalling tunes cPRC responses to UV/violet stimulation**

To investigate how NO signalling alters the dynamics of the cPRC circuit, we carried out calcium imaging experiments.
We ubiquitously expressed the calcium sensor GCaMP6s in larvae and imaged calcium signals during 405 nm stimulation of the cPRCs.
As we have shown previously, a 20-sec local stimulation of cPRC cilia lead to a transient increase in cPRC calcium levels, followed by a transient decrease [@Veraszto2018].
After \~20-sec, calcium levels in cPRCs were raising again, reaching higher levels than at the start of the stimulus -- a response that may involve depolarisation (Figure 4A, B).
This activation phase occurs after the 20 sec stimulation period and is likely due to a delayed neuroendocrine feedback [@Veraszto2018].
To determine whether NO mediates such a feedback, we repeated the experiment in *NOS*-mutant larvae.
While we detected the initial activation phase followed by inhibition, in homozygous *NOS*-mutants for both CRISPR alleles this was not followed by delayed activation.
Instead, calcium levels dropped to a low steady-state level (Figure 4A, B).
We thus identified a requirement for NO signalling in the late-phase activation of cPRCs.

**Two unconventional guanylyl cyclases are expressed in the cPRCs**

We aimed next to identify the NO-receptor in the cPRCs.
NO generally acts via soluble guanylate cyclases (sGC), belonging to the guanylate cyclase family with a CYC domain (PFAM domain: PF00211).
NO binding to the heme group of sGC leads to increased cyclic guanosine monophosphate (cGMP) production.
Analysis of sGCs in *Platynereis* indicated that these genes are not expressed in any of the cells of the cPRC circuit [@Veraszto2017].
Recently, Moroz and coworkers reported an atypical but widely conserved family of guanylyl cyclases with a NIT (nitrite/nitrate sensing) domain (PF08376) (NIT-GC) as potential mediators of NO signalling [@Moroz2020].
To identify NIT-GCs in *Platynereis*, we searched transcriptome resources and retrieved 15 potential NIT-GC homologs (Figure 4---figure supplement 1 and 2).
To analyse the relationship of these sequences to metazoan NIT-GCs, we retrieved protein sequences with a CYC domain from the transcriptome and genome databases of 45 metazoan and 2 choanoflagellate species.
We carried out cluster analysis and did phylogenetic reconstruction on a group of membrane-bound guanylyl cyclases with sGCs as an outgroup.
In agreement with Moroz et al. [@Moroz2020], we found a group of GCs with NIT domains with representatives in placozoans, cnidarians, some ecdysozoans, echinoderms, and lophotrochozoans.
The 15 *Platynereis* sequences belonged to several deeply diverged clades in the phylogenetic tree (Figure 4---figure supplement 1 and 2).
To characterise the expression of NIT-GCs, we used previously published spatially mapped single-cell transcriptome data [@Achim2015; @williams2017].
Among the 15 NIT-GCs, two showed high and specific expression in the cPRCs and one was expressed in the INNOS cells (Figure 4---figure supplement 2).
In the single-cell data, we could identify the cPRCs by the specific expression of *c-opsin1* and the *pedal-peptide2 neuropeptide precursor* (*MLD proneuropeptide*), previously described cPRC markers [@Arendt2004; @williams2017] (Figure 4---figure supplement 3A).
The INNOS cells were identified by NOS expression and spatial mapping in the brain [@Achim2015].
We decided to focus on two NIT-GCs expressed in the cPRCs and with a full-length sequence, *NIT-GC1* and *NIT-GC2*.
To confirm the single-cell data, we first carried out in situ hybridisation chain reaction (HCR) with probes for *NIT-GC1* and *NIT-GC2* mRNA. Both genes were specifically expression in the four cPRCs, as confirmed by co-labeling with an acetylated α-tubulin antibody and with an HCR probe against *pedal peptide 2/MLD proneuropeptide* (Figure 4C, D and Figure 4---figure supplement 3A-C).
To analyse the subcellular localisation of NIT-GC1 and NIT-GC2 at the protein level, we raised and affinity-purified polyclonal antibodies against a specific peptide sequence from both proteins.
In immunostainings, we found that NIT-GC1 was localise to the region corresponding to the axonal projections of the cPRCs in the anterior nerve plexus (Figure 4E).
Co-immunostaining with the rabbit NIT-GC1 antibody and a custom rat antibody raised against *Platynereis* NOS revealed the localisation of both proteins in dots in the neurosecretory neuropil (Figure 4---figure supplement 3D).
In contrast, NIT-GC2 specifically labelled the ramified sensory cilia of the cPRCs (Figure 4F).
These different subcellular localisations suggest that the two NIT-GCs are involved in different intracellular signalling processes in the ciliary and axonal regions of the cPRCs.

**NIT-GC1 produces cGMP in an NO-dependent manner**

To further characterise these atypical guanylyl cyclases, we focused on NIT-GC1 and carried out in vitro experiments.
In bacteria, NIT domains are thought to regulate cellular functions in response to intra- or extracellular nitrate and nitrite.
NIT-GC1 has a NIT domain and a highly conserved cyclase domain that is expected to catalyse cGMP synthesis (Figure 4G).
The NIT domain may render NIT-GC1 dependent on NO signals.
To test this, we co-expressed the cGMP indicator Green cGull [@Matsuda2017] and NIT-GC1 in cultured COS-7 (monkey kidney) cells, a cell line with minimal endogenous sGC activity.
For balanced expression, we used a single plasmid with the two open-reading frames separated by the 2A self-cleaving peptide (Figure 4H).
Application of the NO donor SNAP lead to increased cGMP levels, an effect we did not observe when cells were exposed to DMSO or when Green cGull was expressed alone (Figure 4I-K).
To test whether cGMP production is dependent on the NIT domain, we also tested a deletion construct of NIT-GC1 lacking the NIT domain (Figure 4G).
Cells expressing this construct and Green cGull did not show increased cGMP levels when exposed to SNAP (Figure 4L).
These results indicate that NIT-GC1 is able to catalyse cGMP production in an NO-dependent manner and this function requires the NIT domain.
These results establish NIT-GC1 as a biochemical NO sensor.

**NIT-GC1 is required for NO-mediated retrograde signalling to cPRCs during the UV response**

To test the in vivo function of NIT-GC1 and NIT-GC2 in cPRC responses, we combined calcium imaging with morpholino-mediated knockdowns.
We used two translation-blocking morpholinos for each *NIT-GC* gene and tested knockdown efficiency by immunostaining injected animals with the NIT-GC1 and NIT-GC2 antibodies (Figure 4---figure supplement 3E,F).
For both genes, the morpholinos led to a strong reduction in the respective antibody signal, confirming efficient knockdown and antibody specificity.

In NIT-GC1 morphant larvae, the delayed activation of cPRCs following 405 nm stimulation did not occur (Figure 4M).
This phenotype is similar to the phenotype of *NOS* mutants suggesting that NIT-GC1 acts as the NO sensor in cPRCs to drive their delayed activation.
This could occur via increased cGMP production and the opening of a cyclic-nucleotide-gated (CNG) channel specific to cPRCs [@Tosches2014].
NIT-GC2 morphant larvae, in contrast, showed a step-up increase in calcium following light stimulation (Figure 4N).
The calcium signal decayed during stimulation and was off after light off.
These data support an essential role for ciliary-localised NIT-GC2 in suppressing cPRC calcium following its transient rise at stimulus onset.
Overall, these knockdown experiments revealed different signalling mechanisms for the two NIT-GCs that may be due to their different subcellular localisations.

```{r echo=FALSE,fig.cap="**Figure 4. NOS and two NIT-GCs shape calcium signals during cPRC UV/violet response.** **(A, B)** GCaMP6s signals in cPRCs in wild type and *NOS* mutant (A, NOSΔ11/Δ11, B, NOSΔ23/Δ23) larvae during 405 nm light stimulation. **(C, D)** In situ HCR for (C) *NIT-GC1* and (D) *NIT-GC2* (magenta) in three-day-old *Platynereis* larvae. Larvae were co-stained with an antibody agains acetylated α-tubulin to label cPRC cilia and the neuropil (green). **(E, F)** Immunostaining for (E) NIT-GC1 and (F) NIT-GC2 (magenta), co-stained for acetylated α-tubulin (green). **(G)** The domain structure of *Platynereis* NIT-GC1 and the truncated NIT-GC1ΔNIT protein lacking the NIT domain. A predicted transmembrane region (TM) is shown in grey. **(H)** Schematic of the cell-based assay to detect cGMP production following the addition of an NO donor SNAP or DMSO as control. **(I-L)** Green cGull fluorescence over time for the four conditions tested. Individual responses and their mean with 0.95 confidence interval are shown (n > 6 cells). Intensities are normalized (ΔF/F0). The indicated chemicals were added at 2 min after the start of imaging (grey bars). **(M, N)** GCaMP6s signals in cPRCs in (M) NIT-GC1 and (N) NIT-GC2 morphant larvae during 405 nm light stimulation. Individual responses and their mean with 0.95 confidence interval are shown."}

knitr::include_graphics(path = "figures/Fig4.png")

```

**NO signalling shapes the dynamics of the cPRC circuit**

To investigate how NO and NIT-GC signalling influence the dynamics of the cPRC circuit, we imaged calcium signals from postsynaptic neurons in wild type, mutant and morphant larvae.
We were able to image the activity of all neurons in the cPRC circuit (INNOS, INRGW, Ser-h1 and MC).
The MC cell was identified based on its position and intrinsic activity [@Veraszto2017].
To unambiguously identify all other cells from which we recorded calcium signals, we developed an on-slide immunostaining method (Figure 5---figure supplement 1A).
We used the cell-specific antibody markers against RYamide (INNOS) (Figure 5---figure supplement 1B-D), RGWamide (INRGW) and serotonin (Ser-h1) [@Conzelmann2011] to immunostain agar-embedded larvae following calcium imaging.
Based on the position of the nuclei, we could correlate live and fixed samples at a single-cell precision (Figure 5A,B).
Due to the stereotypy of the larvae, we could also identify neurons based on their position and calcium activity in activity-correlation maps (Figure 5C).

We first quantified the responses of the INNOS and INRGW interneurons during 405 nm stimulation of the cPRCs.
In both wild type and *NOS*-mutant larvae, INNOS cells showed an increase in calcium during stimulation (Figure 5D).
In contrast, the INNOS response was flat or slightly negative in NIT-GC2 morphant larvae (Figure 5E) revealing an essential role for NIT-GC2-mediated cPRC suppression in INNOS activation.
INRGW cells were initially inhibited during cPRC stimulation, followed by a delayed activation paralleling the second activation phase of cPRCs.
This late INRGW response was lacking in *NOS*-mutants (Figure 5F).
In NIT-GC2 morphants, INRGW cells showed a transient increase in calcium that decayed after light off and a delayed activation was not present (Figure 5G).
Next, we imaged calcium signals from Ser-h1 and MC neurons in wild type and *NOS* mutant larvae.
Ser-h1 cells showed an activation profile that correlated with cPRC activity, including a reduction in calcium during stimulation followed by rebound, a response that was defective in *NOS* mutants (Figure 5I).
MC cells showed sustained activation, including a late-phase that was lacking in *NOS* mutants (Figure 5I).
These data suggest that during 405 nm stimulation the Ser-h1 cells are inhibited and MC cells are activated, and this regulation is NO-dependent.
This pattern is expected to inhibit ciliary activity in the prototroch but not in the other ciliary bands, triggering NO-dependent downward swimming.

```{r echo=FALSE,fig.cap="**Figure 5. NOS- and NIT-GC2-dependent dynamics of the cPRC circuit.** **(A, B)** GCaMP6s imaging from cPRCs and INNOS cells (left panels) followed by on-slide immunostaining for (A) RYamide to label INNOS and (B) RGWamide+serotonin to label INRGW and Ser-h1 (red). Nuclei are stained with DAPI (cyan). Asterisks indicate cPRC nuclei. Numbers mark the same cells in the GCaMP and immunostaining images matched by position. **(C)** Correlation map of neuronal activity of the cPRCs, INNOS, INRGW, Ser-h1 and MC neurons. **(D)** GCaMP6s fluorescence in INNOS cells in wild type (WT) and NOSΔ11/Δ11 mutant larvae during 405 nm stimulation of the cPRC cilia. **(E)** GCaMP6s fluorescence in INNOS cells in NIT-GC2 morphant larvae during 405 nm stimulation. **(F)** GCaMP6s fluorescence in INRGW cells in wild type and NOSΔ23/Δ23 mutant larvae during 405 nm stimulation. **(G)** GCaMP6s fluorescence in INRGW cells in NIT-GC2 morphant larvae during 405 nm stimulation. **(H, I)** GCaMP6s fluorescence in (H) Ser-h1 cells and (I) the MC cell in wild type and NOSΔ11/Δ11 mutant larvae during 405 nm stimulation."}

knitr::include_graphics(path = "figures/Fig5.png")

```

**Mathematical modelling of cPRC-circuit dynamics**

To further analyse the dynamics of responses to UV light and formally describe cPRC phototransduction, synaptic connections and NO retrograde signalling, we developed a mixed cellular-circuit-level mathematical model. We used our Ca2+ imagining data of cPRC, INNOS and INRGW cells collected in wild type, *NOS* knockout, and *NIT-GC2* morphant larvae to formulate assumptions that are the basis of the proposed model.

We model a direct c-opsin1-dependent response to UV leading to the initial rise in Ca2+ levels. The mechanism of this signal is not known but may include Gα and Gβ signalling, CNG channel activation or other pathways. We further assume that in cPRC cells, Ca2+ levels increase with cGMP. We model a decrease in cGMP and Ca2+ levels due to a NIT-GC2-dependent pathway. The effect of NO in cPRC cells is captured by another term that describes a NIT-GC1 and NO-dependent increase in cGMP. Based on the synaptic connectome, we infer a feedforward coupling between cPRC cells and INNOS and INRGW cells. The proposed synaptic coupling is UV-dependent, decays linearly and is suppressed in NIT-GC2 morphants.

We analysed the neurotransmitter-related genes expressed in the respective cells identified from the single-cell transcriptome data of Achim et al. to understand the neurotransmission mechanisms in cPRC, INNOS and INRGWa [@Achim2015; @williams2017].
Interestingly, in cPRC, whereas cholinergic neurons has previously been implicated in neurotransmission [@Randel2014a], we found transporters and synthetic enzymes involved in glutaminergic, glycinergic, GABAergic and adrenergic innervation, respectively (Figure 6A).
Gene expression has been observed by Randel et al. in these genes, suggesting that they may be expressed in cPRC [@Randel2014a].
Their receptors were also found for each of the expressed genes within INNOS and INRGWa (Figure 6A).
Based on these expression patterns and results, a model of transduction across the cPRC pathway focusing on Ca2+, cGMP and NO dynamics is proposed (Figure 6B).
The c-opsin1, which is UV-sensitive, activates G proteins (Gi/Go) and decreases cGMP CNGAα, which is already known to be strongly expressed in cPRC as one of the targets highly dependent on cGMP changes, may regulate the influx of ions into the cell [@Arendt2004; @tsukamoto2017; @Veedin2021].
The Gβγ released by Gi/o activation also induces K+ efflux (hyperpolarisation) via GIRK channels and closes the voltage-gated Ca2+ channels, thereby reducing intracellular calcium levels[@tsukamoto2023].
It is also thought that increased calcium levels in INNOS signalled from cPRC may activate NOS and generate nitric oxide; INNOS has glycinergic and glutamatergic potential and may act as an inhibitory signal regulator for INRGWa.
INRGWa, and may function as an inhibitory signal modulator for INRGWa.

We used this model and calcium imaging data to develop a mathematical model.

```{r echo=FALSE,fig.cap="**Figure 6. Signalling mechanisms and mathematical modelling between cPRC, INNOS and INRGWa.** **(A)** Dot plot of genes (columns) expressed in three types of cells (rows) in the cPRC circuit using single cell RNA-Seq. The size of the dots is expressed in proportion to the percentage of cells expressing that gene relative to all cells. The colours represent the normal logarithm of the number of transcripts in the cells expressing the gene. **(B)** Schematic diagram of the signalling pathway of the cPRC circuit, focusing on the NO feedback. **(C)** Diagram for **(D)**"}

knitr::include_graphics(path = "figures/Fig6.png")

```

## Discussion

Our work revealed an essential role for NO-mediated signalling in driving UV/violet avoidance behaviour.
NO, produced by postsynaptic INNOS interneurons, signals retrogradely to the presynaptic cPRCs leading to their delayed activation.
The resulting NIT-GC1-dependent late-phase activation of the photoreceptors drives circuit output through projection interneurons and ciliomotor neurons.
In the cPRC circuit, synaptic connectivity alone is thus not sufficient to drive circuit dynamics and behavioural change.
We identified cPRC-expressed NIT-GC1 as a new type of NO sensor.
NIT-GC1 is localised to the axonal projections of the cPRCs in the anterior neurosecretory plexus of the brain.
Here, cPRC axons intermingle and synapse on INNOS neurites that contain NOS.
NO-mediated retrograde signalling thus likely occurs in the neurosecretory plexus where NOS- and NIT-GC1-containing projections are in close proximity and where we could detect NO production following UV stimulation.
We showed that NIT-GC1 can produce cGMP in a NO-dependent manner.
In cPRCs, increased cGMP levels could open CNGAα, a cyclic nucleotide-gated channels specifically expressed in cPRC [@Tosches2014], leading to their delayed depolarisation, even after UV off.

Also our mathematical model...

**Functional diversity of NIT-GCs**

NO is a free radical with a millisecond-to-second scale half life in vivo and a limited signalling range.
In neuronal signalling, NOS and sGC have been found to localise at close proximity at synapses [@burette2002][@garthwaite2015a].
We identified 12 sGCs in *Platynereis*, but none of these is expressed in the cPRC.
Instead, we identified an unconventional NIT-domain containing GC, NIT-GC1 as the mediator of NO retrograde signalling.
The NIT domain was first identified in bacteria and animal NIT-GCs have only been recently reported [@Shu2003; @Moroz2020].
Bacterial NIT domains regulate cellular functions in response to changes in extracellular and intracellular nitrate and/or nitrite concentrations [@camargo2007].
NO is readily converted to nitrate and nitrite [@santos2011; @moeller2019; @garthwaite2015a] and these molecules accumulate in placozoans and cnidarians in cells and tissues with high NOS activity [@Moroz2004; @Moroz2020].
NIT domains in NIT-GCs may also sense nitrate and nitrite, as in bacteria.
Furthermore, if different NIT-GCs show different sensitivities to NO, nitrite and nitrate, it is possible that differences in the half-lives of these molecules could give rise to a range of activation timings [@lundberg2011].
We have found 15 NIT-GCs in *Platynereis*, suggesting a wide range of functions.
In addition, NIT-GC1 and NIT-GC2 showed very different subcellular localisation and functions.
It is not clear whether the functional differences are due to the different localisation or also to differences in biochemical function.
Overall, the diversification and spatially different localisation patterns of NIT-GCs may increase their signalling repertoire.
Our data revealed key roles for two *Platynereis* NIT-GCs in phototransduction and NO-mediated neuromodulation suggesting that these molecules contribute to the diversity of neuronal signalling.
Sponges and ctenophores, where NO signalling is present, lack NIT-GCs.
In placozoans as many as 12 have been found, and cnidarians also contain a large diversity.
Soluble GCs are expressed broadly in the *Drosophila* and mammalian brain and the specificity of NO signalling seems to depend on localised NOS expression and activation, and ectopic NOS expression can confer NO-dependent effects [@Aso2019].
In *Platyneries* larvae

**Nitric oxide confers short-term memory to circuit activity**

Why does a seemingly roundabout feedback pathway exist?
The activity of synaptic circuits is richly modulated monoamines, neuropeptides and gaseous molecules.
The gaseous second messenger nitric oxide (NO) is known to activate a post- to presynaptic retrograde signalling cascade involving cyclic guanosine monophosphate (cGMP) production.
It has been reported that NO produced in one of the mammalian retinal cells modulates cGMP-mediated neurotransmission by activating the presynaptic nerve sGC (a general target of NO) through retrograde signalling [@Vielma2014; @Wei2012].
Retrograde signalling of NO is also known to be one of the molecules that regulate memory formation.
In Drosophila, NOS of PPL1-γ1pedc shortens memory retention while promoting fast memory updating in response to new experiences, indicating that NO regulates forgetting [@Aso2019].
In the ellipsoid body of the *Drosophila* central brain, NOS is specifically expressed in the R3 ring neuron, but not other ring (R) neurons [@Kuntz2017]. R3 neurons are required for visual working memory with NO signalling mediating a short-term (~4 sec) component of the visual memory of objects. In this circuit, NO produced in R3 axons acts in an autocrine manner to activate sGC in R3 axons leading to a CNG-dependent (cyclic nucleotide-gated ion channel) temporary increase in calcium levels [@Kuntz2017]. Because of the short life-time of NO, this molecule may be well suited to encode transient memory traces [@Kuntz2017].

In this study, our mathematical modelling results focused on the time of UV stimulation and showed that the late activity comes more delayed during short UV and sooner during long UV.
This suggests that animals may have a UV length-dependent short-term memory system that regulates cPRC activation, and that there may be mechanisms by which UV stimulation is enhanced and maintained such that NO feedback maintains neural activation even with slight UV.
It is possible that these mechanisms may have been early signalling pathways controlling stimulus changes across space and time in animals' short-term memory.
In planarian UV avoidance behaviour, different time and length scales of neuropeptide and small molecule transmission were reported to generate incoherent patterns of neural activity that competitively control behaviour and memory [@bray2023].
Short-term memory is essential for locating less exposed areas to minimise UV-induced damage in animals.

**The mystery of why IN-RGW activation causes downward swimming**

One type of interneuron that strongly synapses with the cPRC is INRGWa, which synapses directly onto ciliary motoneurons and is therefore considered to be a very important neuron in larval behavioural control [@williams2017; @Veraszto2018].
Our results suggest that the depolarising response of the cPRC by NO feedback induces INRGW activation.
Our experiments have shown that exposure of larvae to RGWamide results in simultaneous downward swimming and that activation of INRGW is sufficient to trigger UV-avoidance behaviour (unpublished data).
However, whole-cell tracking by connectome has not revealed any equilibrium organ that recognises the direction of gravity, and it remains to be investigated how activation of INRGWs causes animals to recognise 'down', e.g. in RGWamide-deficient larvae.
It is known that complex integrative networks of neurons are involved in UV avoidance behaviour in planaria [@Shettigar2021] and Drosophila [@Imambocus2022].
It has been reported that the sensory neurons required for UV avoidance behaviour in Drosophila converge and arrest neuromodulatory hub neurons, and that compartmentalised circuit organisation and neuropeptide release from the regulatory hubs act as central circuit elements controlling escape responses.
RGW interneurons have been suggested to be important for relaying peptidergic signals due to the distinct anatomy and connectivity of projection neurons in peptidergic networks and their high authority values, and central with many neuronal inputs and outputs network components may function as regulatory hubs that gate the activation of specific networks [@williams2017].
The finding of sGCs in INRGWs from the present single-cell analysis also suggests that they may be regulated by NO.
The specific demarcation of sensory input and output through the use of neurotransmitters, neuropeptides and even gaseous NO may further increase the efficiency of network computation in hub neurons and may lead to a more complex mechanism of 'down' recognition.

## Key resources table

```{r echo=FALSE}
library(readxl)
key_resource_tale <- read_excel("key_resource_tale/key_resource_tale.xlsx")
df_key_resource_tale <- as.data.frame(key_resource_tale)
knitr::kable(df_key_resource_tale, format = "html", booktabs = TRUE)
```

## Materials and Methods

**CRISPR-Cas9 Design and Microinjection**

Before designing the small guide RNA (sgRNA) for the sgRNA:Cas9 nuclease, splice sites and polymorphic sites in our laboratory culture were identified to avoid them.
The sgRNA targeted the third exon of Platynereis dumerilii NOS (target site: 5'-GGGCAATACTGGCTCCACTC-3').
The sgRNA was assembled from two annealed oligonucleotides (5'-TAGGGCAATACTGGCTCCACTC-3', 5'-AAACGAGTGGAGCCAGTATTGC-3') forming overhangs for cloning into a BsaI site of the plasmid pDR27456 (Hwang et al. 2013)(42250, Addgene), which contains next to the BsaI site a tracrRNA sequence.
The plasmid was then used to PCR amplify DNA (primers: T7, 5'-AAAAGCACCGACTCGGTGCC-3') for synthesizing the sgRNA.
The DNA was purified with the QIAquick PCR Purification Kit (Qiagen).
From the DNA, the sgRNA was synthesized with the MEGAshortscript Kit (Thermo Fisher Scientific) and was purified with the MEGAclear Kit (Thermo Fisher Scientific).
Cas9-mRNA was transcribed, capped, and polyA-tailed with the mMessage mMachine Kit and the Poly(A) Tailing Kit (both Thermo Fisher Scientific) from a plasmid (pUC57-T7-RPP2-Cas9) containing the Cas9 ORF fused to 169 base pair 5' UTR from the Platynereis dumerilii 60S acidic ribosomal protein P2.
The sgRNA (18 ng/ml) and the Cas9-mRNA (180 ng/µl) were coinjected into fertilized eggs of Platynereis dumerilii wild-type parents according to an established injection procedure [@Conzelmann2013a].
The eggs were kept at 18°C for 45 min before injection and were injected at 14.5°C.
The injected individuals were kept at 18°C for 5 to 8 days in 6-well- plates (Nunc multidish no. 150239, Thermo Scientific) and then cultured at 22°C until sexual maturity.
The mature worms were crossed to wild-type worms and the progeny was genotyped, resulting in two founder lines, which were bred to homozygosity.

**NOS sequencing and genotyping**

For genotyping of the NOS locus, genomic DNA was isolated from single larvae, groups of 6-20 larvae, or from the tails of adult worms.
The DNA was amplified by PCR (primers: 5'-GGTTCATTGGTTTCGATAACATTGCGG-3', 5'-CAGAGTCGATCAGTCTGCATATCTCCA-3') with the dilution protocol of the Phusion Human Specimen Direct PCR Kit (Thermo Scientific).
The PCR product was sequenced directly with a nested sequencing primer (5'-GGTGCTCTCCCGGGTACACAA-3').
A mixture of wild-type and deletion alleles in a sample gave double peaks in the sequencing chromatograms, with the relative height of the double peaks reflecting the relative allele ratio in the sample.

**Vertical column setup for measuring photoresponses**

Photoresponses of larvae of different ages were assayed in a vertical Plexiglas column (31 mm x 10 mm x 160 mm water height).
The column was illuminated from top with light from a monochromator (Polychrome II, Till Photonics).
The monochromator was controlled by AxioVision 4.8.2.0 (Carl Zeiss MicroImaging GmbH) via analog voltage.
The light passed a collimator lens (LAG-65.0-53.0-C with MgF2 Coating, CVI Melles Griot) before entering the column.
The column was illuminated from both sides with light-emitting diodes (LEDs).
The LEDs on each side were grouped into two strips.
One strip contained UV (395 nm) LEDs (SMB1W-395, Roithner Lasertechnik) and the other infrared (810 nm) LEDs (SMB1W-810NR-I, Roithner Lasertechnik).
The UV LEDs were run at 4 V to stimulate the larvae in the column from the side.
The infrared LEDs were run at 8 V (overvoltage) to illuminate the larvae for the camera (DMK 22BUC03, The Imaging Source), which recorded videos at 15 frames per second and was controlled by IC Capture (The Imaging Source).

**Comparing behavior of wildtype and NOS-knockout 3-day-old larvae**

To compare the behavior of wildtype and NOS-knockout larvae at 3 days in the vertical column, the larvae were mixed and left in the dark for 5 min.
The larvae were treated with NOS inhibitors for pharmacology.
The NOS inhibitors were L-NAME.
The larvae were treated with different concentrations in adjacent columns.
The concentrations for the NOS inhibitors were control, 1 mM, 0.1 mM.
The larvae were recorded for 1 min in the dark followed by exposure to collimated cyan (480 nm) light from the top of the column for 2 min, then 2 min darkness, and finally collimated UV (395 nm) light from the top of the column for 2 min.
Stimulus light was provided by the monochromator (Polychrome II, Till Photonics).
Scripts are available at <https://github.com/JekelyLab/NOS>.

**NOS Identification and Phylogenetic Analysis**

To identify NOS, we obtained a "seed" database of oxygenase domain in Pfam database, PF02898.
From these sequences, we produced a Hidden Markov Model (HMM) and used this to mine the 47 metazoan species, 2 choanoflagellate species and 2 filasterea species investigated.
HMM models were run in HMMR3 with an e-value of 1e−15.
We ran CD-Hit [@Fu2012] to eliminate redundant sequences (at a 80% threshold).
We aligned the sequences with MAFFT version 7, with the iterative refinement method E-INS-i.
Alignments were trimmed with TrimAl in gappy-out mode [@Capella-Gutiérrez2009].
To calculate maximum-likelihood trees, we used IQ-tree2 with the LG+G4 model.
To calculate branch support, we ran 1,000 replicates with the aLRT-SH-like and aBayes methods [@Minh2020].
The sequences used for the phylogenetic analysis are available in **Supplementary File 1**, the trimmed alignment is available in **Supplementary File 2** and the pre-trimmed data in **Supplementary file 3**.

**NIT-GC Identification and Phylogenetic Analysis**

To identify NIT-GCs, we obtained a "seed" database of Adenylate and Guanylate cyclase catalytic domain in Pfam database, PF00211.
From these sequences, we produced a Hidden Markov Model (HMM) and used this to mine the 45 metazoan species, 2 choanoflagellate species and 2 filasterea species investigated.
HMM models were run in HMMR3 with an e-value of 1e−15.
We ran CD-Hit [@Fu2012] to eliminate redundant sequences (at a 80% threshold).
To identify clusters, we used the convex-clustering option with 100 jack-knife replicates.
The NIT-GCs are extremely well conserved in membrane-bound guanylate cyclases and form an easily recognizable cluster.
To analyze the phylogeny of NIT-GCs, the cluster containing these GCs together with membrane-bound guanylate cyclases were parsed and used for tree building.
We aligned the sequences with MAFFT version 7, with the iterative refinement method E-INS-i.
Alignments were trimmed with TrimAl in gappy-out mode [@Capella-Gutiérrez2009].
To calculate maximum-likelihood trees, we used IQ-tree2 with the LG+G4 model.
To calculate branch support, we ran 1,000 replicates with the aLRT-SH-like and aBayes methods [@Minh2020].
The sequences used for the phylogenetic analysis are available in **Supplementary File 1**, the trimmed alignment is available in **Supplementary File 2** and the pre-trimmed data in **Supplementary file 3**.

**Single-cell analysis**

We used Achim et al. for the single-cell data [@Achim2015].
In Williams et al., they used 107 cells as neurons by removing duplicates from Achim et al. single-cell data, so we used those cells [@williams2017].
Since the raw data were read count data, we normalized them to TPM using Python.
After that we converted them to log10.
From the sum of the expression levels in 107 cells for each gene, We calculated the percentage in each cell.
For each cell, we identified them with marker genes.
After created the data in Python, plotted it using R dot plots.
RPKM calculates the total number of reads per million bp and then divides by the length of each gene, so it is not possible to compare between samples.
Instead, TPM first divides by the length of the gene and then divides by the total number of reads per million bases, which allows for more accurate comparisons between samples.
In this case we wanted to compare between samples, so we used TPM.
The total TPM of each gene between the samples was used to calculate the percentage of expressed genes.
The total TPM values for each gene and the percentage of expressed genes are available in **Supplementary file 1**.

**In situ HCR**

Larvae were fixed and treated with Proteinase K, according to the conventional WMISH protocol [@Tessmar-Raible2005], with fixation in 4% paraformaldehyde/ PTW (PBS with 0.05% Tween20) for 2 hr at room temperature, and Proteinase K treatment in 100 µg/ml Proteinase K/ PTW for 3 min [@Tessmar-Raible2005].
Specifically, for the HCR protocol, samples were processed in 1.5 ml tubes.
Probe hybridization buffer, probe wash buffer, amplification buffer, and fluorescent HCR hairpins were purchased from Molecular Instruments (Los Angeles, USA).
Hairpins associated with the b2 initiator sequence were labeled with Alexa Fluor 647, and the hairpins associated with the b3 initiator sequence were labeled with Alexa Fluor 546.
To design probes for HCR, we used custom software [@Kuehn2022] to create 20 DNA oligo probe pairs specific to P. dumerilii NOS, NIT-GC1, NIT-GC2, RYa-pNP (GenBank accession: JF811330.1), and MLD/pedal 2-pNP (GenBank accession: KF515945.1).
The NOS, NIT-GC1 and NIT-GC2 probes were designed to be associated with the b2 initiator sequence, while the RYa-pNP and MLD/pedal 2-pNP probes were designed to be associated with the b3 initiator sequence.
For the detection stage, samples were pre-hybridized in 200 µl of probe hybridization buffer for 1 hr at 37°C, and then incubated in 250 µl hybridization buffer containing probe oligos (4 pmol/ml) overnight at 37°C.
To remove excess probe, samples were washed 4× with 1 ml hybridization wash buffer for 15 min at 37°C, and subsequently 2× in 1 ml 5× SSCT (5× SSC with 0.1% Tween20) for 5 min at room temperature.
For the amplification stage, samples were pre-incubated with 100 µl of amplification buffer for 30 min, room temperature, and then incubated with 150 µl amplification buffer containing fluorescently labeled hairpins (40nM concentration (2ul of 3uM stock in 150ul amplification buffer, snap-cooled as described; [@Choi2018]) overnight in the dark at 25°C.
To remove excess hairpins, samples were washed in 1 ml 5× SSCT at room temperature, twice for 5 min, twice for 30 min, and once for 5 min.
During the first 30 min wash, samples were counterstained with DAPI (Cat. #40043, Biotium, USA).

**Immunohistochemistry**

Whole-mount immunostaining of 2 day old Platynereis larvae fixed with 4% paraformaldehyde were carried out using primary antibodies raised against NIT-GC1, NIT-GC2, NOS, RYamide neuropeptide, RGWamide neuropeptide in rabbit, plus a commercial antibody raised against acetylated tubulin in mouse (Sigma T7451).
The synthetic peptides contained an N-terminal Cys that was used for coupling during purification.
Antibodies were affinity purified from sera as previously described [@Conzelmann2012].
Immunostainings were carried out as previously described [@Conzelmann2012].
The NOS promoter (fragment sizes: 12 Kb) was amplified and cloned upstream of 3xHA- Palmi-tdTomato.
Larvae injected with promoter constructs (ca. 250 ng/ml) were analysed for reporter expression at 3 days post fertilization using an AxioImager Z.1 fluorescence wide-field microscope (Carl Zeiss GmbH, Jena) and immediately fixed for immunostainings.
The protocol followed for immunostaining of HA-tagged reporters was recently described [@Veraszto2017].
Specimens were imaged with a LSM 780 NLO or LSM 880 with Airysan Confocal Microscope (Zeiss, Jena).

**Calcium imaging**

For calcium imaging, 49--55 hpf larvae were used.
Experiments were performed at room temperature and larvae were immobilised by being embedded in 2.5% agarose filtered artificial seawater between a slide and coverslip spaced with adhesive tape.
GCaMP6s mRNA (1 mg/ml) was injected into zygotes as described previously [@Randel2014a].
Larvae were imaged on a Zeiss LSM 880 with Airyscan (with a C-Apochromat 63X/1.2 Corr - water) with a frame rate of 1.88 frame/sec and an image size of 512 x 512 pixels.
The larvae were stimulated in a region of interest (a circle with 50 pixel diameter) with 405 nm lasers controlled by the Bleaching mode.
The imaging laser had a similar intensity than the stimulus laser but covered an area that was 10 times larger than the stimulus ROI.

**Cell culture experiment**

Green cGull was used for the cGMP assay [@Matsuda2017].
A full-length Pdum-NIT-GC1 and -NIT-GC2 coding sequences were amplified by PCR starting from a *Platynereis dumerilii* cDNA library and cloned into the pcDNA3.1(+) vector using the T2A self-cleaving sequence.
Cos-7 cells with low expression of endogenous soluble guanylate cyclase were used as cultured cells for gene expression.
This cell line was purchased from Angio-proteomie (CAT no. cAP-0203).
The Cos-7 cells were maintained at 37 °C in 35mm dishes (Nunc™ Glass Bottom Dishes) containing 3 mL of DMEM, high glucose glutamax medium (Thermo; Cat. No. 10566016) supplemented with 10% fetal bovine serum (Thermo; Cat. No. 10082147).
Upon reaching confluency of approximately 85%, we transfected the cells with the plasmid containing Green cGull-T2A-NITGC1.
Transfections were carried out with 150 ng of each plasmid and 0.3 μl of the transfection Lipofectamine 3000 Reagent (invitrogen; Cat. No. L3000001).
Two days post-transfection, we removed the culture medium and substituted it for fresh DMEM-medium.
For single-wavelength imaging experiments, cells in 35-mm dishes were washed twice and imaged in modified Ringer's buffer (140 mM NaCl, 3.5 mM KCl, 0.5mM NaH2PO4 , 0.5mM S-3 MgSO4 , 1.5 mM CaCl2 , 10 mM HEPES, 2 mM NaHCO3 and 5 mM glucose).
Dishes were mounted on a stage heated at 37 °C and imaging was performed using an inverted microscope (LSM880, Zeiss) equipped with an oil-immersion objective lens (UApo/340, 40, NA = 0.17).
Images were acquired using a xenon lamp, 460--495 nm excitation filter, 505-nm dichroic mirror and 510-- 550-nm emission filter (Zeiss).
S-Nitroso-N-acetyl-D, L-penicillamine (SNAP) was purchased from Sigma-Aldrich (St. Louis, MO, USA) .
The exposure time of the EM-CCD camera was controlled by the ZEN software (Zeiss).
Images were acquired every 15 s for 10 min and stimulation was initiated 2 min after starting image acquisition.
Imaging data analysis was performed using ImageJ (National Institutes of Health, Bethesda, MD, USA).

## Acknowledgements

This work was funded by the Wellcome Trust (214337/Z/18/Z).
This project has received funding from the European Research Council (ERC) under the European Union's Horizon 2020 research and innovation programme (grant agreement No 101020792).
KJ has been supported by a JSPS Postdoctoral fellowhip, LAYG by a BBSRC Discovery fellowship (BB/W010305/1).

## References {.unnumbered}

::: {#refs}
:::

## Figure supplements

```{r echo=FALSE,fig.cap="**Figure 1 -- figure supplement 1**  Phylogenetic tree of NOS by maximum likelihood. Tree robustness was tested with 1000 replicates of ultrafast bootstrap with the aLRT-SH-like and aBayes methods."}
knitr::include_graphics(path = "figures/Fig1_sup1.png")

```

```{r echo=FALSE,fig.cap="**Figure 1 -- figure supplement 2** Expression analysis of the *NOS* gene (magenta) using in situ HCR using the larva at three-day-old in dorsal **(A)** and anterior view (B). (B) Insert: showing two NOS cells (INNOS_dl and INNOS_vl) on left side."}
knitr::include_graphics(path = "figures/Fig1_sup2.png")

```

```{r echo=FALSE,fig.cap="**Figure 1 -- figure supplement 3** ssTEM reconstruction of the INNOS (A), INRGW (B) and cPRC (C), anterior view. NS plexus; neurosecretory plexus"}
knitr::include_graphics(path = "figures/Fig1_sup3.png")

```

```{r echo=FALSE,fig.cap="**Figure 3 -- figure supplement 1** **(A)** Top: The domain and Exon/Intron structure of NOS. Bottom: Close-up region showing the genomic locus of NOS gene and the wild-type sequence (WT) targeted by CRISPR/Cas9. The generated mutants (NOSΔ11, NOSΔ23) are also shown. Pink indicates target sites. Gray shows PAM sequences, red shows stop codons. **(B)** Overlaid trajectories for WT (n=37) and NOS mutant (NOSΔ11/Δ11, n=18 and NOSΔ23/Δ23, n=8) at two-day-old larvae. 0 sec as the starting point. After 10 sec, UV (395 nm) stimulation from the side. **(C)** The temporal changes in the vertical position of the WT and mutant two-days-old larvae before and after UV stimulation are shown. The starting points of each larval trajectory are set to 0. After UV stimulation is indicated by purple squares. **(D, E)** The temporal changes in the distance traveled of the WT and mutant in two-day (D) and three-day-old (E) larvae before and after UV stimulation are shown. **(F)** The temporal changes in the distance traveled of larvae treated with NOS inhibitors, L-NAME in three-day larvae before and after UV stimulation are shown. **(G)** Vertical swimming in wild-type (WT) and mutant (NOSΔ11 and NOSΔ23) larvae at two-day-old stimulated with UV (395 nm) light from side, blue (488 nm) light from top and UV (395 nm) light from top. The data are shown in 30 s bins."}
knitr::include_graphics(path = "figures/Fig3_sup1.png")

```

```{r echo=FALSE,fig.cap="**Figure 4 -- figure supplement 1** Cluster analysis of guanylate and adenylate cyclases. Connections are based on blast similarities < 1e-16 as shown on the upper right. Animal groups are colour coded as shown on the upper left. NIT-GCs, NIT domain containing guanylate cyclases; membrane-bound GCs, Membrane-bound guanylyl cyclases; sGCs, soluble guanylate cyclases; ACs, adenylate cyclases."}
knitr::include_graphics(path = "figures/Fig4_sup1.png")

```

```{r echo=FALSE,fig.cap="**Figure 4 -- figure supplement 2** Phylogenetic tree of guanylate cyclase by maximum likelihood (ML). Guanylate cyclase-coupled receptor and soluble guanylate cyclases (sGC) as outgroups. Guanylate cyclases with NIT domains are found in most animal phyla except Porifera, Ctenophora, Urochordata and Chordata. Dot plot of Platynereis NIT-GC genes (columns) expressed in cPRC, INNOS and INRGWa (rows) using single cell RNA-Seq. The size of the dots is expressed in proportion to the percentage of cells expressing that gene relative to all cells. The colours represent the normal logarithm of the number of transcripts in the cells expressing the gene."}
knitr::include_graphics(path = "figures/Fig4_sup2.png")

```

```{r echo=FALSE,fig.cap="**Figure 4 -- figure supplement 3** **(A)** Co-expression analysis image of the NIT-GC1 (magenta) and MLD-pedal2 amide proneuropeptide gene (MLD: green). Anterior view of the larva at two-day-old. **(B, C)** Expression analysis of the NIT-GC2 gene (magenta) using in situ HCR. Anterior (B) and posterior (C) views of the larva at three-day-old. **(D)** Co-localisation analysis using NIT-GC1 (magenta) and NOS (green) antibodies. Anterior view of the larva at two-day-old. **(E, F)** Localisation analysis using NIT-GC1 and NIT-GC2 antibodies for NIT-GC1 (E) and NIT-GC2 (F) morphant. Green shows co-staining with acetylated α-tubulin antibody (acTub). Anterior view of the larva at two-day-old."}
knitr::include_graphics(path = "figures/Fig4_sup3.png")

```

```{r echo=FALSE,fig.cap="**Figure 5 -- figure supplement 1** **(A)** Schematic diagram of the immunostaining procedure after calcium imaging.**(B-D)** Co-expression analysis image of the NOS (magenta) and RY amide proneuropeptide gene (RYa: green). Anterior view of the larva at two-day-old. "}
knitr::include_graphics(path = "figures/Fig5_sup1.png")

```
