---
title: Nitric-oxide-feedback to ciliary photoreceptor cells orchestrates UV avoidance
  in zooplankton
author: "Kei Jokura, Nobuo Ueda, Martin Gühmann, Luis Alfonso Yañez-Guerra, Gáspár Jékely"
date: '*Living Systems Institute, University of Exeter, Stocker road, Exeter, EX4
  4QD, UK*'
output:
  pdf_document:
    latex_engine: lualatex
    template: default
  html_document:
    theme: lumen
  word_document: default
geometry: left=0.8in,right=0.8in,top=0.8in,bottom=0.5in
mainfont: Arial
monofont: Times New Roman
documentclass: article
classoption: onecolumn
linestretch: 1.2
fontsize: 10pt
bibliography: references.bib
csl: elife.csl
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)

```

## Abstract

Nitric oxide (NO) generated by nitric-oxide synthase (NOS) is a key regulator of animal physiology. In nervous systems, NO can modulate circuits, but its effects on neural dynamics and behaviour remain unclear. Here we show how NO signalling shapes circuit dynamics to orchestrate light-avoidance behaviour. We studied UV-avoidance mediated by brain ciliary photoreceptor cells (cPRCs) in larval *Platynereis dumerilii*, a marine annelid. We found NOS specifically expressed in interneurons (INNOS) postsynaptic to cPRCs. Stimulation of cPRCs by violet light leads to cPRC inhibition but concomitant INNOS activation and NO production. NO feeds back to cPRCs and triggers delayed cPRC activation through an unconventional guanylyl cyclase. This results in the activation of a second population of postsynaptic interneurons (INRGW). In *NOS* knockout larvae, NO feedback and INRGW activation do not occur and avoidance behaviour is defective. Our results reveal how NO-mediated neuroendocrine signalling gates a synaptic circuit to initiate light-avoidance behaviour.

## Introduction

The ability to perceive changes in the environment and adapt behaviour accordingly is essential for the survival of all living organisms. Avoidance from UV radiation, which damages DNA essential for survival, is proposed to be one of the behaviours that existed before the abundant development of Earth's oxygen to block UV radiation in the atmosphere (Pittendrigh). In the ocean, almost all zooplankton have a pronounced rhythmic behaviour called Diel vertical migration (DVM), which consists of upwards towards the surface at dusk and descend back to deeper water before dawn. Avoidance behaviour to escape harmful UV radiation during the day is closely linked to the control of vertical migration in the ocean. In metamorphosis, a life history strategy widely found in metazoans, downward swimming towards the seafloor is known to be associated with larval settlement and the initiation of metamorphosis, a dramatic body change from the larval stage, and UV-avoidance behaviour allows dispersal and the acquisition of different ecological niches. However, little is known about the molecular neural circuit mechanisms in UV avoidance behaviour in many animals, including zooplankton. In neural circuits in general light response, the activation of various receptors by excitatory and inhibitory neurotransmitters such as glutamate, GABA and dopamine is known in various species. In addition to this, it has long been known that in nitric oxide (NO), as a neuromodulatory molecule, it is also important as a modulator of visual activity. NO is a small free-radical molecule with diffusible and changeable characteristics, and thus works as one of the neural signalling molecules. The function of NO in the neural circuitry of vision has been well studied in vertebrates. In the mammalian retina, NO is an essential signalling molecule in all retinal cells. NO produced by nitric oxide synthase (NOS) in amacrine cells, one of the retinal cells, activates soluble guanylate cyclase (sGC) expressed in cone bipolar cells, which produces cGMP and regulates downstream signalling. We therefore hypothesise that NO-mediated neural signalling may be involved in the photoreceptor circuits that control UV avoidance behaviour. To test this hypothesis we used the annelid *Platynereis dumerilii*, which conserved vertebrate-type ciliary photoreceptors (cPRCs) [@Arendt2004]. The cPRC expresses c-opsin1, which absorbs UV radiation, is involved in melatonin production and regulates circadian swimming behaviour [@Tosches2014; @tsukamoto2017]. Furthermore, we previously reported abnormal UV avoidance behaviour in c-opsin1 knockout larvae [@Veraszto2018]. The cPRCs express the UV-absorbing c-opsin1 and mediate UV avoidance in the larvae [@Veraszto2018]. In adult worms, UV light and c-opsin1 modulate locomotion and maturation [@Veedin2021]. These suggest that Platynereis larvae are a highly suitable model for elucidating the neural circuit mechanisms of NO-induced UV avoidance behaviour, and the accessibility of genome editing, behavioural assays, calcium imaging and whole-body connectomics could be very useful for study at the molecular level[@ozpolat2021].

## Results

**Nitric oxide synthase is expressed in interneurons of the UV-avoidance circuit**

We identified a single *nitric oxide synthase* (*NOS*) gene in the *Platynereis dumerilii* genome and transcriptome data. Phylogenetic analysis of NOS proteins indicate that the *Platynereis* NOS belongs to an orthology group of metazoan NOS sequences <!---# need to add a NOS tree as supplement -->. To characterise the expression pattern of *NOS* we used in situ hybridization chain reaction (HCR) and transient transgenesis. In two-day-old larvae, we detected *NOS* expression in two cells in the apical organ region (Fig. 1A) <!---# 3d HCR and transgene? -->. The same two cells cells were labelled with a reporter construct driving palmitoylated-tdTomato by a 12 kbp upstream genomic region of the *NOS* gene. This reporter also revealed the axonal projections of the NOS-expressing neurons. In three-day-old larvae... <!---# need 3d HCR and transgene data as well -->

The position and morphology of the four *NOS*-expressing cells allowed us to identify the same cells in the three-day-old whole-body volume EM data [@williams2017; @veraszto2020] as four interneurons (INNOS)(Figure 1D-E). The INNOS cells are postsynaptic to the UV-sensory cPRCs and presynaptic to the INRGW interneurons, which are also cPRC targets (Fig. 1F).

![**Figure 1. Identification of NOS-expressing interneurons (INNOS) within the cPRC circuit.** (A and B) Immunostaining with anti-HA antibody against HA tagged palmitoylated-tdTomato expressed under the upstream of the start site of NOS (magenta). Co-staining with acetylated α-tubulin antibody as marker for cilia and axonal scaffold (ac α-tub: green). INNOSs with the entire cell body and axon labelled are shown with white arrowheads and with only the cell body labelled are shown with black arrowheads. Views of larvae at 2-day-old (A) and 3-day-old (B) from the anterior side, respectively. (C) Three types of neurons (cPRC, INNOS and INRGWa) in the cPRC circuit reconstructed in the whole-body transmission electron microscopy (ssTEM) dataset of a 3-day-old larva. The INNOS on the ventral side labelled with (A) and (B) are shown in purple. Other neurons are shown in light grey. Axons and dendrites appear as lines and cell body positions are represented by spheres. (D and E) Expression analysis of the NOS gene (magenta) using in situ HCR. Anterior view of the larva at 2-day-old. (D) Co-stained image using acetylated α-tubulin antibody (ac α-tub: green). (E) Co-expression analysis image with the RYamide proneuropeptide gene (RYa-pNP: green), which is highly expressed in INNOS. White arrowheads indicate co-expressed INNOS. Black arrowheads indicate larval eyes. White dashed lines indicate the outer frame of the larva. (F) Wiring diagram of the cPRC circuit. Hexagons represent neuronal groups (numbers indicated in square brackets), arrows represent synaptic connections (numbers indicated in number of synapses); NOS and RGW indicate INNOS and INRGW-dc1 respectively. Ciliomotor neuron shows two serotonergic neurons (Ser-h1).](../figures/Fig1.png){width="100%"}

**Nitric oxide signalling mediates UV-avoidance behaviour** The specific expression of NOS in interneurons directly postsynaptic to the cPRCs suggests that NO signalling is involved in the UV response. To test this, we generated two *Platynereis* *NOS* knockout lines with the CRISPR/Cas9 method. We recovered two deletions (NOSΔ11/Δ11 and Δ23/Δ23), both leading to an early stop codon and thus likely representing null alleles (Figure 3A). We could establish a homozygous line for both mutations indicating that NOS is not an essential gene. To quantify UV avoidance, we recorded the trajectories of freely swimming larvae in vertical columns, illuminated laterally from two opposite sides with xy nm violet light. As previously shown, wild-type larvae swim downward within 30 sec following UV/violet light stimulation. In contrast, homozygous *NOS*-mutant larvae showed a strongly diminished UV-avoidance response (Figure 3B). This phenotype is similar to the defective UV-avoidance of *c-opsin1* mutant larvae and reveals a requirement for NOS in UV-avoidance behaviour.

![**Figure 2. Larvae with inhibited NOS have shorter ciliary arrest times and a delayed response to UV-avoidance.** (A) Percentage of time in which cilia are closed in DMSO-treated larvae (control) vs. larvae exposed to NOS inhibitor. (B) Top: The domain structure of NOS. Bottom: Close-up region showing the genomic locus of NOS gene and the wild-type sequence (WT) targeted by CRISPR/Cas9. The generated mutants (NOSΔ11, NOSΔ23) are also shown. Pink indicates target sites. Yellow shows PAM sequences, red shows stop codons. (C and D) Vertical swimming in wild type and mutant larvae at 2-day-old (C) and 3-day-old (D) stimulated with blue (480 nm) and UV (395 nm) light from above. the data are shown in 30 s bins.](../figures/Fig2.png){alt="Figure 2. Larvae with inhibited NOS have shorter ciliary arrest times and a delayed response to UV-avoidance. (A) Percentage of time in which cilia are closed in DMSO-treated larvae (control) vs. larvae exposed to NOS inhibitor. (B) Top: The domain structure of NOS. Bottom: Close-up region showing the genomic locus of NOS gene and the wild-type sequence (WT) targeted by CRISPR/Cas9. The generated mutants (NOSΔ11, NOSΔ23) are also shown. Pink indicates target sites. Yellow shows PAM sequences, red shows stop codons. (C and D) Vertical swimming in wild type and mutant larvae at 2-day-old (C) and 3-day-old (D) stimulated with blue (480 nm) and UV (395 nm) light from above. the data are shown in 30 s bins." width="100%"}

**Nitric oxide produced during UV/violet stimulation gates the output of the cPRC circuit**

To test if NO is produced during UV/violet responses, we injected zygotes with the fluorescent NO-reporter DAF-FM and imaged two-day-old larvae during the local stimulation of the cPRC cilia by 405 nm light. Following light stimulation, we detected an increase in DAF-FM fluorescence in the region of the INNOS cells. (#Kei - are there time course data that could be shown?). We next analysed the dynamics of the cPRC circuit following light stimulation. As we have shown previously [@Veraszto2018], a 30-sec illumination of the cPRC ciliary region lead to a decrease in cPRC calcium levels, likely due to the hyperpolarisation of the cPRCs. After 30 min, calcium levels in cPRCs were raising again, reaching higher levels than at the start of the stimulus - a response that may involve depolarisation. This activation phase occurs after the end of the stimulation and may be due to a slower neuroendocrine feedback [@Veraszto2018]. To determine whether NO mediates such a feedback, we measured calcium signals in *NOS*-mutant larvae. While we detected the inhibition phase in cPRC in *NOS*-mutants, this was not followed by activation and calcium levels remained low. These results revealed an essential requirement for NOS signalling in the second phase of the cPRC response.

We next quantified the responses of the INNOS and INRGW interneurons following a UV/violet stimulus to the cPRCs. To unambiguously identify the cells from which we recorded calcium signals, we developed an on-slide immunostaining method. We used the cell-specific neuropeptide markers anti-RYamide (INNOS) and anti-RGWa (INRGW) [@Conzelmann2011] to mark the interneurons. Based on the position of the nuclei, we could correlate live and fixed samples at a single-cell precision (Figure 2 - figure supplement ). In both wild type and *NOS*-mutant larvae, INNOS cells showed increase calcium signals during the inhibition phase of the cPRCs. In contrast, calcium signals increased in INRGW cells in wild-type larvae during the second activation phase of cPRCs, a response that was lacking in *NOS*-mutants. This suggests that there is no circuit output to the Ser-h1 cells and the ciliary band in *NOS*-mutants, consistent with the observed behavioural defects.

**Two unconventional guanylyl cyclases (NIT-GC) are expressed in the cPRCs**

NO generally acts via cyclic guanosine monophosphate (cGMP) as a second messenger. In this signalling pathway, NO binds to the heme group of soluble guanylate cyclase (sGC), a member of the guanylate cyclase family with a CYC domain (PF00211), to increase cGMP. However, no NO targets were found in cells that highly express sGC within the cPRC circuit. Recently, Moroz and others reported that GC with NIT (nitrite/nitrate sensing) domains (PF08376) (NIT-GC), identified as a sensor for nitrate and nitrite, the oxides of NO, is relatively widely conserved in metazoans (Moroz et al., 2020). To search for NIT-GCs in Platynereis, genes with the CYC domains were searched using the transcriptome and genome sequences of 45 metazoan species and choanoflagellate. By sequence-similarity searches, multiple sequence alignments and cluster analyses, a cluster of membrane-bound (guanylate cyclase-coupled receptor) type was found and a phylogenetic tree was constructed with soluble type (sGC) as an outgroup. As in the phylogenetic analysis of Moroz et al, we found a group of GCs with NIT domains that included many metazoans such as placozoa, cnidaria, ecdysozoa, echinoderms and lophotrochozoa, and in Platynereis 15 NIT-GCs were found (Fig. 3A). To find the candidate NIT-GCs involved in the cPRC circuit among the 15 NIT-GCs, we used previously published spatially mapped single-cell transcriptome data [@Achim2015][@williams2017] and Two NIT-GCs were found to be highly expressed in the cPRC, and we analysed their expression using in situ HCR as NIT-GC1 and NIT-GC2. As a result, strong signals for both were observed specifically in the vicinity of the cPRCs in 2-day-old larvae (Fig. 3B). Both c-opsin1 and MLD pNP (pedal peptide neuropeptide precursor 2), already identified as cPRC markers, and NIT-GC1 and NIT-GC2 were found to be co-expressed. They were co-expressed c-opsin1 and MLD pNP (pedal peptide neuropeptide precursor 2), already identified as cPRC markers (Fig. 3C) (Arendt et al., 2004; Williams et al., 2017). To analyse localisations at the protein level, immunostaining was performed using polyclonal antibodies against NIT-GC1 and NIT-GC2. Interestingly, NIT-GC1 was found to localise to the neuronal axon and NIT-GC2 to the ciliary region of the cPRC (Fig. 3D and E). Furthermore, co-immunostaining with NOS antibodies revealed that NOS and NIT-GC1 were localised in close proximity in the neurosecretory plexus (Fig. 3F). These results suggest that NIT-GC1 may function as a major target for NO.

![**Figure 3. cPRC-specific expression of NIT-GC1 localises in close proximity to NOS in the ANS region.** (A) Phylogenetic tree of guanylate cyclase by maximum likelihood (ML). Guanylate cyclase-coupled receptor and soluble guanylate cyclase as outgroups. Guanylate cyclases with NIT domains are found in most animal phyla except Porifera, Ctenophora, Urochordata and Chordata. Tree robustness was tested with 1000 replicates of ultrafast bootstrap. Species names are as follows: Ap (Aiptasia pallida), Bb (Branchiostoma belcheri), Cv (Crassostrea virnigica), Dm (Drosophila melanogaster), Dp (Daphnia pulex), Hh (Hoilungia hongkongensis), Lg (Lottia gigantea), Pc (Priapulus caudatus), Pd (Platynereis dumerilii), Sk (Saccoglossus kowalevskii), Sp (Strongylocentrotus purpuratus), Ta (Trichoplax adhaerens), Tc (Tribolium castaneum). (B-D) Expression analysis of the NIT-GC1 gene (magenta) using in situ HCR. White arrowheads indicate cells expressing NIT-GC1. (B) Co-staining image using acetylated α-tubulin antibody (ac α-tub: green). (C) Co-expression analysis image of the c-opsin1 gene (green). Black arrowheads indicate cells expressing c-opsin1 differently from cPRC. (D) Co-expression analysis image of the MLD-pedal2 amide proneuropeptide gene (MLD-pNP: green). Black arrowheads indicate cells different from cPRC. The white dashed lines in (C) and (D) indicate the outer frame of the larvae. (E, F) Localisation analysis using NIT-GC1 and NIT-GC2 antibodies. Green shows co-staining with acetylated α-tubulin antibody (ac α-tub). (G) Co-localisation analysis using NIT-GC1 and NOS antibodies. (B-G) Anterior view of the 2-day-old larvae.](../figures/Fig3.png){width="100%"}

**NIT-GC1 produces cGMP by nitric oxide**

NIT-GC1 has a highly conserved cyclase domain and activation of NIT-GC1 is expected to produce cGMP. Recently, a synthetic protein called Green cGull was developed as a cGMP indicator that can be used in cultured cell systems (Matsuda et al., 2017). We co-expressed Green cGull and NIT-GC1 in cultured cells COS-7 (monkey kidney) and tested the intracellular dynamics of cGMP in the activation of NIT-GC1 (Fig. 4A). First, S-nitroso-N-acetyl-DL-penicillamine (SNAP; NO donor), 8-Br-cGMP (8-Bromoguanosine 3',5'-cyclic monophosphate; membrane-permeable cGMP analogue, positive control) and DMSO (negative control) were applied to COS-7 cells expressing Green cGull only. 8-Br-cGMP increased fluorescence intensity by more than sixfold, while little changes were detected for SNAP and DMSO (Fig. S4A). Next, the two kind of genes, Green cGull and NIT-GC1, were co-expressed using the 2A self-cleaving peptide and the same solutions were applied. As a result, a clear increase in cGMP fluorescence intensity was detected in SNAP compared to DMSO or expression of Green cGull only, although not as high as the increase in 8-Br-cGMP (Fig. 4B, C). Finally, expression of a mutant form of NIT-GC1 with deletion of the NIT domain significantly suppressed the increase in cGMP (Note, we need re-experiment!, Fig. 4D). These results indicate that NIT-GC1 is activated by NO to produce cGMP. They also show that the NIT domain is required for the activation of NIT-GC1.

![**Figure 4. NIT-GC1 is induced by NO to produce cGMP.** (A) Top: The domain structure of NIT-GC1. Grey squares indicate the transmembrane domain. Bottom shows a schematic of the NIT-GC1 assay; COS7 cells were used to express NIT-GC1 and the cGMP indicator Green cGull, and the NO donor SNAP and DMSO as control. Green cGull shows cGMP-dependent green fluorescence. (B) The intensity changes in cGMP fluorescence over time for the three conditions are shown. Gray traces show the optical responses to individual intensities (n = 6). The individual intensities are normalized (ΔF/F0). The dark line shows the mean value and the band the standard error range of the mean. The respective solution was acted on 2 min after the start of imaging (grey bars). (C) The mean intensity (Fmean/F0) after acting on the solution. Multiple comparisons were made using the Tukey honestly significant difference test.](../figures/Fig6.png){width="100%"}

**NO induces depolarisation of the cPRCs via NIT-GC1**

To determine the function of NO in the cPRC circuit, the intracellular dynamics of neurons in the cPRC circuit were investigated, focusing on NIT-GC1. First, we investigated the changes in NO during UV stimulation of the cPRC using DAF-FM, a fluorescent probe that can detect NO, in order to determine whether NO is produced by UV light. By injecting DAF-FM directly into zygotes, INNOS cells could be identified under live imaging. To visualise the location of cPRCs, the calcium indicator RGECO1a was also injected together, as cPRCs maintain a higher resting calcium levels compared to other cells. Direct UV stimulation to the cPRCs increased the fluorescence intensity of DAF-FM compared to stimulation of non-cPRC region (Fig. 5A). Next, we did calcium imaging of cells in the cPRC circuit during UV stimulation. We locally stimulated the cPRC cilia in wild-type with a 405 nm laser (UV) and detected a slight increase followed by a strong decrease in calcium levels (hyperpolarisation) (Fig. 5B). Turning off the UV after or during the decrease caused a strong increase (depolarisation) and a return to the same calcium level as before (Fig. 5B). These responses were similar to those reported previously [@Veraszto2018]. We performed a similar stimulation using NOS mutants, which hyperpolarised during the UV stimulation, but no depolarising responses were observed when UV was stopped (Fig. 5C). Similarly, in NIT-GC1 knockdown larvae using the morpholinos, no depolarisation after the hyperpolarisation response were observed (Fig. 5D). Interestingly, no hyperpolarising response during UV stimulation was found to occur in NIT-GC2 knockdown larvae. And we compared the changes of the INNOS calcium levels identified by their position in relation to cPRCs during UV stimulation in wild-type and in NOS mutants. In both cases, a depolarising response was observed at the same time as the hyperpolarising response of the cPRCs (Fig. 5E). Furthermore, to perform calcium imaging of INRGWs in UV stimulation, INRGWs were identified by immunostaining with RGWa antibodies after calcium imaging (Fig. 5F). It has been reported that the INRGW cells form many large synapses with synaptic vesicles on two serotonergic ciliomotor neurons (Ser-h1) that are known to play a role in the regulation of ciliary beating and the activity of the INRGW cells increased the negative correlation between the activity patterns of serotonergic neurons and the MC neuron (Williams et al., 2018). The results showed that depolarisation of INNOS occurs at the same time as hyperpolarisation of cPRC in the wild type. Identified INRGW cells were found to cause depolarisation at the same time as cPRC hyperpolarisation in the wild type (Fig. 5G). It was also found that in NOS mutants where cPRC hyperpolarisation does not occur, the INRGW depolarisation response also does not occur (Fig. 5H).

![**Figure 5. NO produced by UV stimulation to cPRCs induces a depolarising response in cPRC via NIT-GC1.** (A) An INNOS visualised by DAF-FM is shown (a white arrowhead). A black arrowhead indicates the position of the adult eye. The white dashed line indicates the outer frame of the larva. All subsequent images are of 2-day-old larvae. (B) Dual-colour imaging of DAF-FM (green) and RGECO1a (magenta) allows simultaneous visualisation of the INNOS and cPRCs. (C) The intensity of DAF-FM in INNOS changes over time when cPRC (top) and other cells (NC) (bottom) are stimulated with UV (405 nm). Hereafter, light purple squares indicate the timing of UV stimulation, gray traces show the optical responses to individual intensities, each intensities are normalized (ΔF/F0) and the dark lines and the bands show the mean value and the standard error range of the mean. (D) Changes over time in GCaMP6s intensities in cPRC when stimulated with UV in WT (purple) and NOS mutant (NOSΔ23) (gray) larvae. (E) Changes over time in GCaMP6s intensities in cPRC when stimulated with UV in control (non-injection) and NIT-GC1 knockdown by morpholino (NITGC1-MO1 and 2) larvae. (F) Co-expression analysis image of the cPRC marker gene (NIT-GC1: magenta) and the INNOS marker gene (RYa-pNP: green) using in situ HCR. White arrowheads indicate INNOS strongly expressing RYa-pNP. (G) INNOS (blue dashed line) identified from calcium imaging using GCaMP6s by their position in relation to the cPRC (purple dashed line). (H) Comparison of changes over time in GCaMP6s intensities in cPRC (purple) and INNOS (blue) when cPRC is stimulated with UV in WT (left) and NOS mutant (NOSΔ11: right) larvae. (I) Schematic diagram of the immunostaining procedure after calcium imaging. (J) The position of the cPRC (purple dashed line) in calcium imaging with GCaMP6s. Cells around the cPRC are numbered according to the position of the nuclei. (K) Immunostaining image using RGWamide (RGWa) and serotonin (Ser) antibodies (red) and DAPI (light blue) in the same region as (J). INRGW (green dashed line) and Ser-h1 (black arrowheads) identified from the numbered nuclear positions are shown. (L) Comparison of changes over time in GCaMP6s intensities in cPRC (purple) and INRGW (green) when the cPRC are stimulated with UV in WT (left) and NOS mutant (NOSΔ23: right) larvae.](../figures/Fig5.png){width="100%"}

## Discussion

In the present study, we explored the role of NO in the UV avoidance behaviour of Platynereis. We demonstrated that there are interneurons expressing NOS in the cPRC circuit and that NO is required for UV avoidance in behavioural experiments using NOS mutants. We identified the axon-localised type NIT-GC1 and the cilia-localised type NIT-GC2 expressed in the cPRCs as targets of NO and showed from cultured cell experiments that these produce cGMP in response to NO. We have shown by calcium imaging that INNOSs are activated in response to the hyperpolarising response of the cPRCs, that the NO produced feeds back to the cPRC via NIT-GC1 and induces a depolarising response in the cPRC, and that the depolarising response of the cPRC is involved in activating downstream INRGW. Lastly, we confirmed that RGWamide induces the downward swimming behaviour of larvae.

cPRCs expressing UV-absorbing c-opsin1 have high resting calcium levels and show hyperpolarising and subsequent depolarising responses to UV light (405 nm) [@Veraszto2018]. In the present study, we have shown that the cells that strongly synapse with cPRCs are interneurons that specifically express NOS (INNOS). The fluorescence intensity of DAF-FM, which specifically detected INNOS, showed a transient tonic-like increase dependent on UV stimulation of the cPRCs. The graph shows a decrease in intensity after UV light is switched off, but the response of DAF-FM to NO is irreversible and this decrease is due to breaching by the relatively strong excitation light. Attempts to measure changes in the intensity of the calcium indicator RGECO1a simultaneously with DAF-FM imaging did not lead to detection of changes in fluorescence intensity compared to GCaMP6s and could only visualise the cPRCs. In calcium imaging using GCaMP6s, INNOS identified by the positional relationship between the cPRC and INNOS and the labelling of RYa depolarised in clear synchrony with the hyperpolarising response of the cPRCs. This tonic response may involve dopaminergic nerves based on single-cell data (Fig) [@puopolo2001][@zhang2007][@Randel2014a]. In the mammalian retina, it is known that signals from bipolar photoreceptor cells are transmitted to downstream NOS-expressing amacrine cells (NOAC), where NO is produced [@jacoby2018]. These suggest that UV stimulation of cPRCs causes the production of NO via a depolarising response of INNOS.

Our results suggest that NIT-GC1, which is specifically expressed in cPRCs, functions as a target for NO produced by INNOS and contributes to the regulation of the cPRC circuit transmission pathway via a depolarising response by increasing cGMP. In the mammalian retina, NO is an essential signalling molecule and one of the most well-studied [@Cudeiro1999]. It has been reported that NO produced in one of the retinal cells modulates cGMP-mediated neurotransmission by activating presynaptic neural sGCs (general target of NO) through retrograde signalling [@Vielma2014][@Wei2012]. In Platynereis cPRC, NIT-GC1 was also identified as having the property of producing cGMP in a NO-dependent manner. Furthermore, it is known that CNGAα, one of the cyclic nucleotide-gated channels, is specifically highly expressed in cPRC [@Tosches2014], and in INNOS-cPRC signalling, NO is implicated in the neuromodulation of the cPRC by retrograde signalling. NO is a molecule with a very short half-life, and when functioning as a signalling molecule, NOS and sGC are known to localise very close together at synapses between neurons [@burette2002][@garthwaite2015a]. In our study, we found 12 sGCs in Platynereis, but none expressed in the cPRC. Localisation analysis of NOS and NIT-GC1 by antibody staining showed that the two proteins localised very close together in the anterior neurosecretory plexus region. Calcium imaging results using NOS mutant and NIT-GC1 knockdown larvae suggest that NIT-GC1 activation induces a depolarising response in the cPRC.

Another type of interneuron that strongly synapses with the cPRC is the INRGWa, which synapses directly onto the ciliomotor neuron and is therefore considered to be a very important neuron in the control of larval behaviour [@Veraszto2018][@williams2017]. The present results suggest that the depolarising response of the cPRC due to NO feedback induces INRGW activation. RGWamide is also a neuropeptide that induces larval downward swimming quite strongly (unpublished? need to show data?), and the delayed downward swimming of NOS mutant larvae in response to UV stimulation may be responsible for the failure of INRGW activation to occur. In addition, in many marine invertebrates, NO has been reported to be involved in many aspects of settlement behaviour and metamorphosis induction [@leise2001; @bishop2003; @bishop2007; @pechenik2007; @castellano2014; @ueda2013; @ueda2014; @ueda2016; @yang2018; @zhu2020; @song2021]. The downward swimming behaviour of Platynereis larvae is the first step in the onset of settlement behaviour and metamorphosis. These suggest that downward swimming due to NO activation of INRGWs may trigger the onset of settlement behaviour and metamorphosis [@Conzelmann2013][@Williams2015]. These suggest that NO feedback signalling may act as a trigger for downward swimming via activation of INRGWs, which induces the initiation of settlement behaviour and metamorphosis.

An outstanding question is how the NO\~NIT-GC response alters neuronal dynamics. NO is a single molecule and, due to its short half-life, cannot be transmitted over long distances and has a limited signalling repertoire. It has therefore been considered unsuitable for generating diversity in signalling compared to other signalling molecules such as neuropeptides and classical neurotransmitters [@Jékely2021]. However, our results in the present study indicate that NO-induced activation of NIT-GC1 is involved in the regulation of neuronal circuits (cPRC pathway). The NIT domain was originally found in bacteria and the existence of GCs with NIT domains was first reported by Moroz et al in 2020 [@Shu2003; @Moroz2020]. It has been proposed that NIT domains in bacteria regulate cellular functions in response to changes in extracellular and intracellular nitrate and/or nitrite concentrations [@camargo2007]. In fact, NO is a free radical molecule, with a half-life of 5 msec in vivo, and is readily converted to nitrate and nitrite [@santos2011; @möller2019; @garthwaite2015a]. And nitrate and nitrite have been reported to accumulate in cells and tissues with high NOS activity in placozoans and cnidarians [@Moroz2004; @Moroz2020]. It is possible that the NIT domains of NIT-GC1 identified in this study may also be used for nitrate/nitrite sensing, as in bacteria. Furthermore, if they show different sensitivities to NO, nitrite and nitrate, it is possible that the temporal variation in their respective half-lives could give rise to a repertoire of activation timings [@lundberg2011]. In addition, we have found at least 15 NIT-GCs in the Platynereis transcriptome data, of which NIT-GC2 is highly expressed in the cPRC, in addition to NIT-GC1. Interestingly, NIT-GC2 showed a very different localisation pattern to NIT-GC1, and furthermore, the results of calcium imaging of cPRC in response to UV stimulation using NIT-GC2 knockdown larvae were the different dynamics (such that hyperpolarisation was suppressed) with NIT-GC1 knockdown. It was not clear from the present analysis whether this difference was due to differences in the localisation or the property of the NIT-GCs. However, the diversification and spatially different localisation patterns of NIT-GCs may increase their signalling repertoire. We also believe that the appearance of NIT-GCs was one of the means to generate the diversity of signalling pathways using NO during the early stages of nervous system evolution in metazoans, since sponges and ctenophores, where NO signalling is present, have no NIT-GCs at all, as many as 12 have been found in placozoans, and NIT-GCs are functional in the "proto-eye" circuit of Platynereis.

![**Figure 6. Gene expressions and signalling mechanisms between cPRC, INNOS and INRGWa.** (A) Dot plot of genes (columns) expressed in three types of cells (rows) in the cPRC circuit using single cell RNA-Seq. The size of the dots is expressed in proportion to the percentage of cells expressing that gene relative to all cells. The colours represent the normal logarithm of the number of transcripts in the cells expressing the gene. (B) Schematic diagram of the signalling pathway of the cPRC circuit, focusing on the NO feedback.](../figures/Figure6.png){width="100%"}

## Materials and Methods

**CRISPR-Cas9 Design and Microinjection**

Before designing the small guide RNA (sgRNA) for the sgRNA:Cas9 nuclease, splice sites and polymorphic sites in our laboratory culture were identified to avoid them. The sgRNA targeted the third exon of Platynereis dumerilii NOS (target site: 5'-GGGCAATACTGGCTCCACTC-3'). The sgRNA was assembled from two annealed oligonucleotides (5'-TAGGGCAATACTGGCTCCACTC-3', 5'-AAACGAGTGGAGCCAGTATTGC-3') forming overhangs for cloning into a BsaI site of the plasmid pDR27456 (Hwang et al. 2013)(42250, Addgene), which contains next to the BsaI site a tracrRNA sequence. The plasmid was then used to PCR amplify DNA (primers: T7, 5'-AAAAGCACCGACTCGGTGCC-3') for synthesizing the sgRNA. The DNA was purified with the QIAquick PCR Purification Kit (Qiagen). From the DNA, the sgRNA was synthesized with the MEGAshortscript Kit (Thermo Fisher Scientific) and was purified with the MEGAclear Kit (Thermo Fisher Scientific). Cas9-mRNA was transcribed, capped, and polyA-tailed with the mMessage mMachine Kit and the Poly(A) Tailing Kit (both Thermo Fisher Scientific) from a plasmid (pUC57-T7-RPP2-Cas9) containing the Cas9 ORF fused to 169 base pair 5' UTR from the Platynereis dumerilii 60S acidic ribosomal protein P2. The sgRNA (18 ng/ml) and the Cas9-mRNA (180 ng/µl) were coinjected into fertilized eggs of Platynereis dumerilii wild-type parents according to an established injection procedure [@Conzelmann2013a]. The eggs were kept at 18°C for 45 min before injection and were injected at 14.5°C. The injected individuals were kept at 18°C for 5 to 8 days in 6-well- plates (Nunc multidish no. 150239, Thermo Scientific) and then cultured at 22°C until sexual maturity. The mature worms were crossed to wild-type worms and the progeny was genotyped, resulting in two founder lines, which were bred to homozygosity.

**NOS sequencing and genotyping**

For genotyping of the NOS locus, genomic DNA was isolated from single larvae, groups of 6-20 larvae, or from the tails of adult worms. The DNA was amplified by PCR (primers: 5'-GGTTCATTGGTTTCGATAACATTGCGG-3', 5'-CAGAGTCGATCAGTCTGCATATCTCCA-3') with the dilution protocol of the Phusion Human Specimen Direct PCR Kit (Thermo Scientific). The PCR product was sequenced directly with a nested sequencing primer (5'-GGTGCTCTCCCGGGTACACAA-3'). A mixture of wild-type and deletion alleles in a sample gave double peaks in the sequencing chromatograms, with the relative height of the double peaks reflecting the relative allele ratio in the sample.

**Vertical column setup for pharmacology**

Freely swimming 2-day-old larvae were recorded in 5 adjacent vertical columns (each 31 mm x 10 mm x 144 mm water height) with a DMK 21BF03 camera (The Imaging Source) at 30 frames/s. The columns were illuminated laterally with red (633 nm) light-emitting diodes (LEDs). This light cannot be detected by 2-day-old larvae. The larvae were treated with NOS inhibitors and NO donors. The NOS inhibitors were L-NAME, SMIS, and AGH. The NO donors were SNAP and NONOate. The larvae were treated with different concentrations in adjacent columns. The concentrations for the NOS inhibitors were control, 10 mM, 1 mM, 0.1 mM, and 0.01 mM; for SNAP were control, DMSO control, 1 mM, 0.1 mM, and 0.01 mM; and for NONOate were control 1 mM, 0.1 mM, and 0.01 mM. The larvae were recorded for 30 s then the drugs were added. 30 s, 1 min, 1 h, and 2 h later, the larvae were recorded again for 30 s. The larvae were tracked and their vertical displacement was calculated. Scripts are available at <https://github.com/JekelyLab/NOS>.

**Vertical column setup for measuring photoresponses**

Photoresponses of larvae of different ages were assayed in a vertical Plexiglas column (31 mm x 10 mm x 160 mm water height). The column was illuminated from top with light from a monochromator (Polychrome II, Till Photonics). The monochromator was controlled by AxioVision 4.8.2.0 (Carl Zeiss MicroImaging GmbH) via analog voltage. The light passed a collimator lens (LAG-65.0-53.0-C with MgF2 Coating, CVI Melles Griot) before entering the column. The column was illuminated from both sides with light-emitting diodes (LEDs). The LEDs on each side were grouped into two strips. One strip contained UV (395 nm) LEDs (SMB1W-395, Roithner Lasertechnik) and the other infrared (810 nm) LEDs (SMB1W-810NR-I, Roithner Lasertechnik). The UV LEDs were run at 4 V to stimulate the larvae in the column from the side. The infrared LEDs were run at 8 V (overvoltage) to illuminate the larvae for the camera (DMK 22BUC03, The Imaging Source), which recorded videos at 15 frames per second and was controlled by IC Capture (The Imaging Source).

**Comparing behavior of wildtype and NOS-knockout 3-day-old larvae**

To compare the behavior of wildtype and NOS-knockout larvae at 3 days in the vertical column, the larvae were mixed and left in the dark for 5 min. The larvae were recorded for 1 min in the dark followed by exposure to collimated cyan (480 nm) light from the top of the column for 2 min, then 2 min darkness, and finally collimated UV (395 nm) light from the top of the column for 2 min. Stimulus light was provided by the monochromator (Polychrome II, Till Photonics). Scripts are available at <https://github.com/JekelyLab/NOS>.

**In situ HCR**

Larvae were fixed and treated with Proteinase K, according to the conventional WMISH protocol [@Tessmar-Raible2005], with fixation in 4% paraformaldehyde/ PTW (PBS with 0.05% Tween20) for 2 hr at room temperature, and Proteinase K treatment in 100 µg/ml Proteinase K/ PTW for 3 min [@Tessmar-Raible2005]. Specifically, for the HCR protocol, samples were processed in 1.5 ml tubes. Probe hybridization buffer, probe wash buffer, amplification buffer, and fluorescent HCR hairpins were purchased from Molecular Instruments (Los Angeles, USA). Hairpins associated with the b2 initiator sequence were labeled with Alexa Fluor 647, and the hairpins associated with the b3 initiator sequence were labeled with Alexa Fluor 546. To design probes for HCR, we used custom software [@Kuehn2022] to create 20 DNA oligo probe pairs specific to P. dumerilii NOS, NIT-GC1, NIT-GC2, RYa-pNP (GenBank accession: JF811330.1), c-opsin1 (GenBank accession: AY692353.1), CNGAα (GenBank accession: KM199644.1), and MLD/pedal2-pNP (GenBank accession: KF515945.1). The NOS, NIT-GC1 and NIT-GC2 probes were designed to be associated with the b2 initiator sequence, while the RYa-pNP, c-opsin1, CNGAα and MLD/pedal2-pNP probes were designed to be associated with the b3 initiator sequence. For the detection stage, samples were pre-hybridized in 200 µl of probe hybridization buffer for 1 hr at 37°C, and then incubated in 250 µl hybridization buffer containing probe oligos (4 pmol/ml) overnight at 37°C. To remove excess probe, samples were washed 4× with 1 ml hybridization wash buffer for 15 min at 37°C, and subsequently 2× in 1 ml 5× SSCT (5× SSC with 0.1% Tween20) for 5 min at room temperature. For the amplification stage, samples were pre-incubated with 100 µl of amplification buffer for 30 min, room temperature, and then incubated with 150 µl amplification buffer containing fluorescently labeled hairpins (40nM concentration (2ul of 3uM stock in 150ul amplification buffer, snap-cooled as described; [@Choi2018]) overnight in the dark at 25°C. To remove excess hairpins, samples were washed in 1 ml 5× SSCT at room temperature, twice for 5 min, twice for 30 min, and once for 5 min. During the first 30 min wash, samples were counterstained with DAPI (Cat. #40043, Biotium, USA).

**Immunohistochemistry**

Whole-mount immunostaining of 2 day old Platynereis larvae fixed with 4% paraformaldehyde were carried out using primary antibodies raised against NIT-GC1, NIT-GC2, NOS, RYamide neuropeptide, RGWamide neuropeptide in rabbit, plus a commercial antibody raised against acetylated tubulin in mouse (Sigma T7451). The synthetic peptides contained an N-terminal Cys that was used for coupling during purification. Antibodies were affinity purified from sera as previously described [@Conzelmann2012]. Immunostainings were carried out as previously described [@Conzelmann2012].

**Calcium imaging**

For calcium imaging, 49--55 hpf larvae were used. Experiments were performed at room temperature and larvae were immobilised by being embedded in 2.5% agarose filtered artificial seawater between a slide and coverslip spaced with adhesive tape. GCaMP6s mRNA (1 mg/ml) was injected into zygotes as described previously (Randel et al., 2014). Larvae were imaged on a Zeiss LSM 880 with Airyscan (with a C-Apochromat 63X/1.2 Corr - water) with a frame rate of 1.88 frame/sec and an image size of 512 x 512 pixels. The larvae were stimulated in a region of interest (a circle with ?? pixel diameter) with 405 nm lasers controlled by the Bleaching mode. The imaging laser had a similar intensity than the stimulus laser but covered an area that was ?? times larger than the stimulus ROI.

**Cell culture experiment**

Green cGull was used for the cGMP assay [@Matsuda2017]. A full-length Pdum-NIT-GC1 and -NIT-GC2 coding sequences were amplified by PCR starting from a Platynereis dumerilii cDNA library and cloned into the pcDNA3.1(+) vector using the T2A self-cleaving sequence. Cos-7 cells with low expression of endogenous soluble guanylate cyclase were used as cultured cells for gene expression. This cell line was purchased from Angio-proteomie (CAT no. cAP-0203). The Cos-7 cells were maintained at 37 °C in 35mm dishes (Nunc™ Glass Bottom Dishes) containing 3 mL of DMEM, high glucose glutamax medium (Thermo; Cat. No. 10566016) supplemented with 10% fetal bovine serum (Thermo; Cat. No. 10082147). Upon reaching confluency of approximately 85%, we transfected the cells with the plasmid containing Green cGull-T2A-NITGC1. Transfections were carried out with 150 ng of each plasmid and 0.3 μl of the transfection Lipofectamine 3000 Reagent (invitrogen; Cat. No. ???). Two days post-transfection, we removed the culture medium and substituted it for fresh DMEM-medium. For single-wavelength imaging experiments, cells in 35-mm dishes were washed twice and imaged in modified Ringer's buffer (140 mM NaCl, 3.5 mM KCl, 0.5mM NaH2PO4 , 0.5mM S-3 MgSO4 , 1.5 mM CaCl2 , 10 mM HEPES, 2 mM NaHCO3 and 5 mM glucose). Dishes were mounted on a stage heated at 37 °C and imaging was performed using an inverted microscope (LSM880, Zeiss) equipped with an oil-immersion objective lens (UApo/340, 40, NA = xx). Images were acquired using a xenon lamp, 460--495 nm excitation filter, 505-nm dichroic mirror and 510-- 550-nm emission filter (????). 8-br-cGMP and S-Nitroso-N-acetyl-D, L-penicillamine (SNAP) were purchased from Sigma-Aldrich (St. Louis, MO, USA) and Cayman Chemical (Ann Arbor, MI, USA). The exposure time of the EM-CCD camera was controlled by the ZEN software (Zeiss). Images were acquired every 15 s for 10 min and stimulation was initiated 2 min after starting image acquisition. Imaging data analysis was performed using ImageJ (National Institutes of Health, Bethesda, MD, USA).

## Acknowledgements

This work was funded by the Wellcome Trust (214337/Z/18/Z).

## References {.unnumbered}

::: {#refs}
:::
